# Drug discovery papers

This repository contains a list of papers on drug discovery.

**Since I do not work on AI for Science, this list does not need to be updated.**  

* **BASC: Applying Bayesian Optimization to the Search for Global Minima on Potential Energy Surfaces**, *ICML*, 2016. [\[pdf\]](http://proceedings.mlr.press/v48/carr16.pdf)   

* **Protein Contact Prediction from Amino Acid Co-evolution using Convolutional Networks for Graph-valued Images**, *NeurIPS*, 2016. [\[pdf\]](https://proceedings.neurips.cc/paper_files/paper/2016/file/2cad8fa47bbef282badbb8de5374b894-Paper.pdf)  

* **Neural Message Passing for Quantum Chemistry**, *ICML*, 2017. [\[pdf\]](https://arxiv.org/pdf/1704.01212.pdf)  

* **Parallel and Distributed Thompson Sampling for Large-scale Accelerated Exploration of Chemical Space**, *ICML*, 2017. [\[pdf\]](https://arxiv.org/pdf/1706.01825.pdf)  

* **Variational Inference for Sparse and Undirected Models**, *ICML*, 2017. [\[pdf\]](https://arxiv.org/pdf/1602.03807.pdf)  

* **Sequence Tutor: Conservative Fine-Tuning of Sequence Generation Models with KL-control**, *ICML*, 2017. [\[pdf\]](https://arxiv.org/pdf/1611.02796.pdf) [\[code\]](https://github.com/magenta/magenta/tree/main/magenta/models/rl_tuner)   

* **Grammar Variational Autoencoder**, *ICML*, 2017. [\[pdf\]](https://arxiv.org/pdf/1703.01925.pdf)  

* **SchNet: A Continuous-filter Convolutional Neural Network for Modeling Quantum Interactions**, *NeurIPS*, 2017. [\[pdf\]](https://arxiv.org/pdf/1706.08566.pdf)    

* **Solid Harmonic Wavelet Scattering: Predicting Quantum Molecular Energy from Invariant Descriptors of 3D Electronic Densities**, *NeurIPS*, 2017. [\[pdf\]](https://proceedings.neurips.cc/paper_files/paper/2017/file/72b386224056bf940cd5b01341f65e9d-Paper.pdf)    

* **Spherical Convolutions and Their Application in Molecular Modelling**, *NeurIPS*, 2017. [\[pdf\]](https://papers.nips.cc/paper_files/paper/2017/file/1113d7a76ffceca1bb350bfe145467c6-Paper.pdf) [\[code\]](https://github.com/deepfold/NIPS2017)  

* **Predicting Organic Reaction Outcomes with Weisfeiler-Lehman Network**, *NeurIPS*, 2017. [\[pdf\]](https://arxiv.org/pdf/1709.04555.pdf) [\[code\]](https://github.com/wengong-jin/nips17-rexgen)       

* **Protein Interface Prediction using Graph Convolutional Networks**, *NeurIPS*, 2017. [\[pdf\]](https://proceedings.neurips.cc/paper_files/paper/2017/file/f507783927f2ec2737ba40afbd17efb5-Paper.pdf) [\[code\]](https://github.com/fouticus/pipgcn)        
* **Learning a Generative Model for Validity in Complex Discrete Structures**, *ICLR*, 2018. [\[pdf\]](https://arxiv.org/pdf/1712.01664.pdf) [\[code\]](https://github.com/DavidJanz/molecule_grammar_rnn)         

* **Syntax-Directed Variational Autoencoder for Structured Data**, *ICLR*, 2018. [\[pdf\]](https://arxiv.org/pdf/1802.08786.pdf) [\[code\]](https://github.com/Hanjun-Dai/sdvae)    

* **Junction Tree Variational Autoencoder for Molecular Graph Generation**, *ICML*, 2018. [\[pdf\]](https://arxiv.org/pdf/1802.04364.pdf) [\[code\]](https://github.com/wengong-jin/icml18-jtnn)        

* **GraphRNN: Generating Realistic Graphs with Deep Auto-regressive Models**, *ICML*, 2018. [\[pdf\]](https://arxiv.org/pdf/1802.08773.pdf) [\[code\]](https://github.com/snap-stanford/GraphRNN)        

* **Generative Modeling for Protein Structures**, *NeurIPS*, 2018. [\[pdf\]](https://proceedings.neurips.cc/paper_files/paper/2018/file/afa299a4d1d8c52e75dd8a24c3ce534f-Paper.pdf)    

* **Constrained Graph Variational Autoencoders for Molecule Design**, *NeurIPS*, 2018. [\[pdf\]](https://arxiv.org/pdf/1805.09076.pdf) [\[code\]](https://github.com/Microsoft/constrained-graph-variational-autoencoder)   

* **Neural Edit Operations for Biological Sequences**, *NeurIPS*, 2018. [\[pdf\]](https://papers.nips.cc/paper_files/paper/2018/file/d0921d442ee91b896ad95059d13df618-Paper.pdf)    

* **Graph Convolutional Policy Network for Goal-Directed Molecular Graph Generation**, *NeurIPS*, 2018. [\[pdf\]](https://arxiv.org/pdf/1806.02473.pdf) [\[code\]](https://github.com/bowenliu16/rl_graph_generation)   

* **Deep Generative Markov State Models**, *NeurIPS*, 2018. [\[pdf\]](https://arxiv.org/pdf/1805.07601.pdf) [\[code\]](https://github.com/markovmodel/deep_gen_msm)   

* **Constrained Generation of Semantically Valid Graphs via Regularizing Variational Autoencoders**, *NeurIPS*, 2018. [\[pdf\]](https://arxiv.org/pdf/1809.02630.pdf)    

* **End-to-end Symmetry Preserving Inter-atomic Potential Energy Model for Finite and Extended Systems**, *NeurIPS*, 2018. [\[pdf\]](https://arxiv.org/pdf/1805.09003.pdf)       

* **Learning Protein Sequence Embeddings using Information from Structure**, *ICLR*, 2019. [\[pdf\]](https://arxiv.org/pdf/1902.08661.pdf) [\[code\]](https://github.com/tbepler/protein-sequence-embedding-iclr2019)   

* **Learning Multimodal Graph-to-Graph Translation for Molecular Optimization**, *ICLR*, 2019. [\[pdf\]](https://arxiv.org/pdf/1812.01070.pdf)    

* **A Generative Model For Electron Paths**, *ICLR*, 2019. [\[pdf\]](https://arxiv.org/pdf/1805.10970.pdf)   

* **Human-level Protein Localization with Convolutional Neural Networks**, *ICLR*, 2019. [\[pdf\]](https://openreview.net/pdf?id=ryl5khRcKm) [\[code\]](https://github.com/ml-jku/gapnet-pl)  

* **Learning Protein Structure with a Differentiable Simulator**, *ICLR*, 2019. [\[pdf\]](https://openreview.net/pdf?id=Byg3y3C9Km)   

* **Molecular Hypergraph Grammar with Its Application to Molecular Optimization**, *ICML*, 2019. [\[pdf\]](http://proceedings.mlr.press/v97/kajino19a/kajino19a.pdf)   

* **Functional Transparency for Structured Data: a Game-Theoretic Approach**, *ICML*, 2019. [\[pdf\]](https://arxiv.org/pdf/1902.09737.pdf)   

* **DeepNose: Using Artificial Neural Networks to Represent the Space of Odorants**, *ICML*, 2019. [\[pdf\]](http://proceedings.mlr.press/v97/tran19b/tran19b.pdf)     

* **Learning to Optimize in Swarms**, *NeurIPS*, 2019. [\[pdf\]](https://openreview.net/pdf?id=rke8aHHlIH) [\[code\]](https://github.com/Shen-Lab/LOIS)  

* **Generative Models for Graph-based Protein Design**, *NeurIPS*, 2019. [\[pdf\]](https://papers.nips.cc/paper_files/paper/2019/file/f3a4ff4839c56a5f460c88cce3666a2b-Paper.pdf) [\[code\]](https://github.com/jingraham/neurips19-graph-protein-design)  

* **Discrete Object Generation with Reversible Inductive Construction**, *NeurIPS*, 2019. [\[pdf\]](https://arxiv.org/pdf/1907.08268.pdf) [\[code\]](https://github.com/PrincetonLIPS/reversible-inductive-construction)  

* **Graph Normalizing Flow**, *NeurIPS*, 2019. [\[pdf\]](https://arxiv.org/pdf/1905.13177.pdf)   

* **Cormorant: Covariant Molecular Neural Networks**, *NeurIPS*, 2019. [\[pdf\]](https://arxiv.org/pdf/1906.04015.pdf) [\[code\]](https://github.com/risilab/cormorant)  

* **N-Gram Graph: Simple Unsupervised Representation for Graphs, with Applications to Molecules**, *NeurIPS*, 2019. [\[pdf\]](https://arxiv.org/pdf/1806.09206.pdf) [\[code\]](https://github.com/chao1224/n_gram_graph)

* **Evaluating Protein Transfer Learning with TAPE**, *NeurIPS*, 2019. [\[pdf\]](https://arxiv.org/pdf/1906.08230.pdf) [\[code\]](https://github.com/songlab-cal/tape)

* **A Model to Search for Synthesizable Molecules**, *NeurIPS*, 2019. [\[pdf\]](https://arxiv.org/pdf/1906.05221.pdf) [\[code\]](https://github.com/john-bradshaw/molecule-chef)

* **Depth-First Proof-Number Search with Heuristic Edge Cost and Application to Chemical Synthesis Planning**, *NeurIPS*, 2019. [\[pdf\]](https://proceedings.neurips.cc/paper_files/paper/2019/file/4fc28b7093b135c21c7183ac07e928a6-Paper.pdf) 

* **End-to-End Learning on 3D Protein Structure for Interface Prediction**, *NeurIPS*, 2019. [\[pdf\]](https://arxiv.org/pdf/1807.01297.pdf) [\[code\]](https://github.com/drorlab/DIPS)  

* **Retrosynthesis Prediction with Conditional Graph Logic Network**, *NeurIPS*, 2019. [\[pdf\]](https://arxiv.org/pdf/2001.01408.pdf) [\[code\]](https://github.com/Hanjun-Dai/GLN)  

* **Symmetry-adapted Generation of 3D Point Sets for the Targeted Discovery of Molecules**, *NeurIPS*, 2019. [\[pdf\]](https://arxiv.org/pdf/1906.00957.pdf) [\[code\]](https://github.com/atomistic-machine-learning/G-SchNet)  

* **Hyperbolic Graph Neural Networks**, *NeurIPS*, 2019. [\[pdf\]](https://arxiv.org/pdf/1910.12892.pdf) [\[code\]](https://github.com/facebookresearch/hgnn)  

* **Deep Imitation Learning for Molecular Inverse Problems**, *NeurIPS*, 2019. [\[pdf\]](https://proceedings.neurips.cc/paper/2019/file/b0bef4c9a6e50d43880191492d4fc827-Paper.pdf)  

* **From Variational to Deterministic Autoencoders**, *ICLR*, 2020. [\[pdf\]](https://arxiv.org/pdf/1903.12436.pdf) [\[code\]](https://github.com/ParthaEth/Regularized_autoencoders-RAE-)  

* **Augmenting Genetic Algorithms with Deep Neural Networks for Exploring the Chemical Space**, *ICLR*, 2020. [\[pdf\]](https://arxiv.org/pdf/1909.11655.pdf) [\[code\]](https://github.com/aspuru-guzik-group/GA)  

* **Oblique Decision Trees from Derivatives of ReLU Networks**, *ICLR*, 2020. [\[pdf\]](https://arxiv.org/pdf/1909.13488.pdf) [\[code\]](https://github.com/guanghelee/iclr20-lcn)  

* **GraphAF: a Flow-based Autoregressive Model for Molecular Graph Generation**, *ICLR*, 2020. [\[pdf\]](https://arxiv.org/pdf/2001.09382.pdf) [\[code\]](https://github.com/DeepGraphLearning/GraphAF)  

* **Model-based Reinforcement Learning for Biological Sequence Design**, *ICLR*, 2020. [\[pdf\]](https://openreview.net/pdf?id=HklxbgBKvr)   

* **InfoGraph: Unsupervised and Semi-supervised Graph-Level Representation Learning via Mutual Information Maximization**, *ICLR*, 2020. [\[pdf\]](https://arxiv.org/pdf/1908.01000.pdf)   
 
* **Directional Message Passing for Molecular Graphs**, *ICLR*, 2020. [\[pdf\]](https://arxiv.org/pdf/2003.03123.pdf)   

* **Energy-based Models for Atomic-resolution Protein Conformations**, *ICLR*, 2020. [\[pdf\]]() [\[code\]](https://github.com/facebookresearch/protein-ebm)  

* **Reconstructing Continuous Distributions of 3D Protein Structure from Cryo-EM Images**, *ICLR*, 2020. [\[pdf\]](https://arxiv.org/pdf/1909.05215.pdf)   

* **RNA Secondary Structure Prediction By Learning Unrolled Algorithms**, *ICLR*, 2020. [\[pdf\]](https://arxiv.org/pdf/2002.05810.pdf) [\[code\]](https://github.com/ml4bio/e2efold)  

* **Retro-star: Learning Retrosynthetic Planning with Neural Guided A-star Search**, *ICML*, 2020. [\[pdf\]](https://arxiv.org/pdf/2006.15820.pdf) [\[code\]](https://github.com/binghong-ml/retro_star)  

* **Generalizing Convolutional Neural Networks for Equivariance to Lie Groups on Arbitrary Continuous Data**, *ICML*, 2020. [\[pdf\]](https://arxiv.org/pdf/2002.12880.pdf) [\[code\]](https://github.com/mfinzi/LieConv)  

* **Population-Based Black-Box Optimization for Biological Sequence Design**, *ICML*, 2020. [\[pdf\]](https://arxiv.org/pdf/2006.03227.pdf)   

* **Learning to Navigate The Synthetically Accessible Chemical Space Using Reinforcement Learning**, *ICML*, 2020. [\[pdf\]](http://proceedings.mlr.press/v119/gottipati20a/gottipati20a.pdf) [\[code\]](https://github.com/99andBeyond/Apollo1060)  

* **Hierarchical Generation of Molecular Graphs using Structural Motifs**, *ICML*, 2020. [\[pdf\]](https://arxiv.org/pdf/2002.03230.pdf)   

* **Multi-Objective Molecule Generation using Interpretable Substructures**, *ICML*, 2020. [\[pdf\]](https://arxiv.org/pdf/2002.03244.pdf) [\[code\]](https://github.com/wengong-jin/multiobj-rationale)  
  
* **A Chance-Constrained Generative Framework for Sequence Optimization**, *ICML*, 2020. [\[pdf\]](http://proceedings.mlr.press/v119/liu20i/liu20i.pdf)   

* **Projective Preferential Bayesian Optimization**, *ICMLR*, 2020. [\[pdf\]](https://arxiv.org/pdf/2002.03113.pdf) [\[code\]](https://github.com/AaltoPML/PPBO)  

* **A Graph to Graphs Framework for Retrosynthesis Prediction**, *ICML*, 2020. [\[pdf\]](https://arxiv.org/pdf/2003.12725.pdf)   

* **A Generative Model for Molecular Distance Geometry**, *ICML*, 2020. [\[pdf\]](https://arxiv.org/pdf/1909.11459.pdf) [\[code\]](https://github.com/gncs/graphdg)  

* **Reinforcement Learning for Molecular Design Guided by Quantum Mechanics**, *ICML*, 2020. [\[pdf\]](https://arxiv.org/pdf/2002.07717.pdf) [\[code\]](https://github.com/gncs/molgym)  

* **Improving Molecular Design by Stochastic Iterative Target Augmentation**, *ICML*, 2020. [\[pdf\]](https://arxiv.org/pdf/2002.04720.pdf) [\[code\]](https://github.com/yangkevin2/icml2020-stochastic-iterative-target-augmentation)  

* **Guiding Deep Molecular Optimization with Genetic Exploration**, *NeurIPS*, 2020. [\[pdf\]](https://arxiv.org/pdf/2007.04897.pdf) [\[code\]](https://github.com/sungsoo-ahn/genetic-expert-guided-learning)  

* **Self-Supervised Graph Transformer on Large-Scale Molecular Data**, *NeurIPS*, 2020. [\[pdf\]](https://arxiv.org/pdf/2007.02835.pdf)     

* **Efficient Generation of Structured Objects with Constrained Adversarial Networks**, *NeurIPS*, 2020. [\[pdf\]](https://arxiv.org/pdf/2007.13197.pdf) [\[code\]](https://github.com/unitn-sml/CAN)  

* **BOSS: Bayesian Optimization over String Spaces**, *NeurIPS*, 2020. [\[pdf\]](https://arxiv.org/pdf/2010.00979.pdf) [\[code\]](https://github.com/henrymoss/BOSS) 

* **Learning Invariances in Neural Networks**, *NeurIPS*, 2020. [\[pdf\]](https://arxiv.org/pdf/2010.11882.pdf) [\[code\]](https://github.com/g-benton/learning-invariances) 

* **Inverse Learning of Symmetries**, *NeurIPS*, 2020. [\[pdf\]](https://arxiv.org/pdf/2002.02782.pdf) [\[code\]](https://github.com/bmda-unibas/InverseLearningOfSymmetries) 

* **BayReL: Bayesian Relational Learning for Multi-omics Data Integration**, *NeurIPS*, 2020. [\[pdf\]](https://arxiv.org/pdf/2010.05895.pdf)  

* **SE(3)-Transformers: 3D Roto-Translation Equivariant Attention Networks**, *NeurIPS*, 2020. [\[pdf\]](https://arxiv.org/pdf/2006.10503.pdf) [\[code\]](https://github.com/FabianFuchsML/se3-transformer-public)  

* **On the Equivalence of Molecular Graph Convolution and Molecular Wave Function with Poor Basis Set**, *NeurIPS*, 2020. [\[pdf\]](https://arxiv.org/pdf/2011.07929.pdf) [\[code\]](https://github.com/masashitsubaki/QuantumDeepField_molecule) 

* **TorsionNet: A Reinforcement Learning Approach to Sequential Conformer Search**, *NeurIPS*, 2020. [\[pdf\]](https://proceedings.neurips.cc/paper/2020/file/e904831f48e729f9ad8355a894334700-Paper.pdf) [\[code\]](https://github.com/tarungog/torsionnet_paper_version)  

* **Goal-directed Generation of Discrete Structures with Conditional Generative Models**, *NeurIPS*, 2020. [\[pdf\]](https://arxiv.org/pdf/2010.02311.pdf)  

* **CogMol: Target-Specific and Selective Drug Design for COVID-19 Using Deep Generative Models**, *NeurIPS*, 2020. [\[pdf\]](https://arxiv.org/pdf/2004.01215.pdf)  

* **Evaluating Attribution for Graph Neural Networks**, *NeurIPS*, 2020. [\[pdf\]](https://papers.nips.cc/paper_files/paper/2020/file/417fbbf2e9d5a28a855a11894b2e795a-Paper.pdf) [\[code\]](https://github.com/google-research/graph-attribution) 

* **Stochastic Normalizing Flows**, *NeurIPS*, 2020. [\[pdf\]](https://arxiv.org/pdf/2002.06707.pdf) [\[code\]](https://github.com/noegroup/stochastic_normalizing_flows) 

* **Barking up the Right Tree: An Approach to Search over Molecule Synthesis DAGs**, *NeurIPS*, 2020. [\[pdf\]](https://arxiv.org/pdf/2012.11522.pdf) [\[code\]](https://github.com/john-bradshaw/synthesis-dags) 

* **Learning Mutational Semantics**, *NeurIPS*, 2020. [\[pdf\]](https://proceedings.neurips.cc/paper_files/paper/2020/file/6754e06e46dfa419d5afe3c9781cecad-Paper.pdf) [\[code\]](https://github.com/brianhie/mutational-semantics-neurips2020) 

* **RetroXpert: Decompose Retrosynthesis Prediction Like A Chemist**, *NeurIPS*, 2020. [\[pdf\]](https://arxiv.org/pdf/2011.02893.pdf) [\[code\]](https://github.com/uta-smile/RetroXpert) 

* **Learning from Protein Structure with Geometric Vector Perceptrons**, *ICLR*, 2021. [\[pdf\]](https://arxiv.org/pdf/2009.01411.pdf) [\[code\]](https://github.com/drorlab/gvp) 

* **MARS: Markov Molecular Sampling for Multi-objective Drug Discovery**, *ICLR*, 2021. [\[pdf\]](https://arxiv.org/pdf/2103.10432.pdf) [\[code\]](https://github.com/bytedance/markov-molecular-sampling) 

* **Practical Massively Parallel Monte-Carlo Tree Search Applied to Molecular Design**, *ICLR*, 2021. [\[pdf\]](https://arxiv.org/pdf/2006.10504.pdf) [\[code\]](https://github.com/yoshizoe/mp-chemts)  

* **Learning Neural Generative Dynamics for Molecular Conformation Generation**, *ICLR*, 2021. [\[pdf\]](https://arxiv.org/pdf/2102.10240.pdf) [\[code\]](https://github.com/DeepGraphLearning/CGCF-ConfGen)  

* **Categorical Normalizing Flows via Continuous Transformations**, *ICLR*, 2021. [\[pdf\]](https://arxiv.org/pdf/2006.09790.pdf) [\[code\]](https://github.com/phlippe/CategoricalNF)  

* **Intrinsic-Extrinsic Convolution and Pooling for Learning on 3D Protein Structures**, *ICLR*, 2021. [\[pdf\]](https://arxiv.org/pdf/2007.06252.pdf) [\[code\]](https://github.com/phermosilla/IEConv_proteins)  

* **HamNet: Conformation-Guided Molecular Representation with Hamiltonian Neural Networks**, *ICLR*, 2021. [\[pdf\]](https://arxiv.org/pdf/2105.03688.pdf)   

* **Property Controllable Variational Autoencoder via Invertible Mutual Dependence**, *ICLR*, 2021. [\[pdf\]](https://openreview.net/pdf?id=tYxG_OMs9WE) [\[code\]](https://github.com/xguo7/PCVAE)  
 
* **Transformer Protein Language Models Are Unsupervised Structure Learners**, *ICLR*, 2021. [\[pdf\]](https://openreview.net/pdf?id=fylclEqgvgd) [\[code\]](https://github.com/facebookresearch/esm)  

* **Symmetry-Aware Actor-Critic for 3D Molecular Design**, *ICLR*, 2021. [\[pdf\]](https://openreview.net/pdf?id=jEYKjPE1xYN)   

* **BERTology Meets Biology: Interpreting Attention in Protein Language Models**, *ICLR*, 2021. [\[pdf\]](https://arxiv.org/pdf/2006.15222.pdf) [\[code\]](https://github.com/salesforce/provis)  

* **Neural Representation and Generation for RNA Secondary Structures**, *ICLR*, 2021. [\[pdf\]](https://arxiv.org/pdf/2102.00925.pdf)   

* **Molecule Optimization by Explainable Evolution**, *ICLR*, 2021. [\[pdf\]](https://openreview.net/pdf?id=jHefDGsorp5) [\[code\]](https://github.com/binghong-ml/MolEvol)  

* **Learning to Make Decisions via Submodular Regularization**, *ICLR*, 2021. [\[pdf\]](https://openreview.net/pdf?id=ac288vnG_7U)   

* **A Structured Observation Distribution for Generative Biological Sequence Prediction and Forecasting**, *ICML*, 2021. [\[pdf\]](http://proceedings.mlr.press/v139/weinstein21a/weinstein21a.pdf)   

* **An End-to-End Framework for Molecular Conformation Generation via Bilevel Programming**, *ICML*, 2021. [\[pdf\]](https://arxiv.org/pdf/2105.07246.pdf) [\[code\]](https://github.com/MinkaiXu/ConfVAE-ICML21)  

* **Fold2Seq: A Joint Sequence(1D)-Fold(3D) Embedding-based Generative Model for Protein Design**, *ICML*, 2021. [\[pdf\]](https://arxiv.org/pdf/2106.13058.pdf) [\[code\]](https://github.com/IBM/fold2seq)  

* **XOR-CD: Linearly Convergent Constrained Structure Generatio**, *ICML*, 2021. [\[pdf\]](http://proceedings.mlr.press/v139/ding21a/ding21a.pdf)   

* **Bayesian Quadrature on Riemannian Data Manifolds**, *ICML*, 2021. [\[pdf\]](https://arxiv.org/pdf/2102.06645.pdf) [\[code\]](https://github.com/froec/BQonRDM)  

* **Improving Molecular Graph Neural Network Explainability with Orthonormalization and Induced Sparsity**, *ICML*, 2021. [\[pdf\]](https://arxiv.org/pdf/2105.04854.pdf) [\[code\]](https://github.com/bayer-science-for-a-better-life/graph-attribution) 

* **LieTransformer: Equivariant Self-Attention for Lie Groups**, *ICML*, 2021. [\[pdf\]](https://arxiv.org/pdf/2012.10885.pdf) [\[code\]](https://github.com/oxcsml/lie-transformer) 

* **Self-Improved Retrosynthetic Planning**, *ICML*, 2021. [\[pdf\]](https://arxiv.org/pdf/2106.04880.pdf)   

* **GraphDF: A Discrete Flow Model for Molecular Graph Generation**, *ICML*, 2021. [\[pdf\]](https://arxiv.org/pdf/2102.01189.pdf)  

* **MSA Transformer**, *ICML*, 2021. [\[pdf\]](http://proceedings.mlr.press/v139/rao21a/rao21a.pdf) [\[code\]](https://github.com/facebookresearch/esm) 

* **Non-Autoregressive Electron Redistribution Modeling for Reaction Prediction**, *ICML*, 2021. [\[pdf\]](https://arxiv.org/pdf/2106.07801.pdf)  

* **E(n) Equivariant Graph Neural Networks**, *ICML*, 2021. [\[pdf\]](https://arxiv.org/pdf/2102.09844.pdf) [\[code\]](https://github.com/vgsatorras/egnn) 

* **Equivariant Message Passing for the Prediction of Tensorial Properties and Molecular Spectra**, *ICML*, 2021. [\[pdf\]](http://proceedings.mlr.press/v139/schutt21a/schutt21a.pdf)  

* **Learning Gradient Fields for Molecular Conformation Generation**, *ICML*, 2021. [\[pdf\]](https://arxiv.org/pdf/2105.03902.pdf) [\[code\]](https://github.com/DeepGraphLearning/ConfGF) 

* **Efficient Generative Modelling of Protein Structure Fragments using a Deep Markov Model**, *ICML*, 2021. [\[pdf\]](http://proceedings.mlr.press/v139/thygesen21a/thygesen21a.pdf)  

* **GEOMOL: Torsional Geometric Generation of Molecular 3D Conformer Ensembles**, *NeurIPS*, 2021. [\[pdf\]](https://arxiv.org/pdf/2106.07802.pdf) [\[code\]](https://github.com/PattanaikL/GeoMol) 

* **Towards Understanding Retrosynthesis by Energy-based Models**, *NeurIPS*, 2021. [\[pdf\]](https://arxiv.org/pdf/2007.13437.pdf)  

* **Learning Graph Models for Retrosynthesis Prediction**, *NeurIPS*, 2021. [\[pdf\]](https://arxiv.org/pdf/2006.07038.pdf) [\[code\]](https://github.com/vsomnath/graphretro)  

* **Hit and Lead Discovery with Explorative RL and Fragment-based Molecule Generation**, *NeurIPS*, 2021. [\[pdf\]](https://arxiv.org/pdf/2110.01219.pdf) [\[code\]](https://github.com/AITRICS/FREED) 

* **GemNet: Universal Directional Graph Neural Networks for Molecules**, *NeurIPS*, 2021. [\[pdf\]](https://arxiv.org/pdf/2106.08903.pdf) [\[code\]](https://www.daml.in.tum.de/gemnet) 

* **A 3D Generative Model for Structure-Based Drug Design**, *NeurIPS*, 2021. [\[pdf\]](https://arxiv.org/pdf/2203.10446.pdf) [\[code\]](https://github.com/luost26/3D-Generative-SBDD) 

* **E(n) Equivariant Normalizing Flows**, *NeurIPS*, 2021. [\[pdf\]](https://arxiv.org/pdf/2105.09016.pdf)   

* **Language Models Enable Zero-shot Prediction of the Effects of Mutations on Protein Function**, *NeurIPS*, 2021. [\[pdf\]](https://proceedings.neurips.cc/paper_files/paper/2021/file/f51338d736f95dd42427296047067694-Paper.pdf) [\[code\]](https://github.com/facebookresearch/esm) 

* **Smooth Normalizing Flows**, *NeurIPS*, 2021. [\[pdf\]](https://arxiv.org/pdf/2110.00351.pdf)  

* **Flow Network based Generative Models for Non-Iterative Diverse Candidate Generation**, *NeurIPS*, 2021. [\[pdf\]](https://arxiv.org/pdf/2106.04399.pdf) [\[code\]](https://github.com/GFNOrg/gflownet) 

* **Multi-Scale Representation Learning on Proteins**, *NeurIPS*, 2021. [\[pdf\]](https://arxiv.org/pdf/2204.02337.pdf)  

* **Representation Learning on Spatial Networks**, *NeurIPS*, 2021. [\[pdf\]](https://proceedings.neurips.cc/paper_files/paper/2021/file/12e35d9186dd72fe62fd039385890b9c-Paper.pdf) [\[code\]](https://github.com/rollingstonezz/SGMP_code) 

* **Predicting Molecular Conformation via Dynamic Graph Score Matching**, *NeurIPS*, 2021. [\[pdf\]](https://proceedings.neurips.cc/paper_files/paper/2021/file/a45a1d12ee0fb7f1f872ab91da18f899-Paper.pdf)  

* **Property-Aware Relation Networks for Few-Shot Molecular Property Prediction**, *NeurIPS*, 2021. [\[pdf\]](https://arxiv.org/pdf/2107.07994.pdf)  

* **Learning Equivariant Energy Based Models with Equivariant Stein Variational Gradient Descent**, *NeurIPS*, 2021. [\[pdf\]](https://arxiv.org/pdf/2106.07832.pdf)  

* **Deep Molecular Representation Learning via Fusing Physical and Chemical Information**, *NeurIPS*, 2021. [\[pdf\]](https://arxiv.org/pdf/2112.04624.pdf)  

* **Motif-based Graph Self-Supervised Learning for Molecular Property Prediction**, *NeurIPS*, 2021. [\[pdf\]](https://arxiv.org/pdf/2110.00987.pdf) [\[code\]](https://github.com/zaixizhang/MGSSL) 

* **Directional Message Passing on Molecular Graphs via Synthetic Coordinates**, *NeurIPS*, 2021. [\[pdf\]](https://arxiv.org/pdf/2111.04718.pdf) [\[code\]](https://www.cs.cit.tum.de/daml/startseite/) 

* **SE(3)-Equivariant Prediction of Molecular Wavefunctions and Electronic Densities**, *NeurIPS*, 2021. [\[pdf\]](https://arxiv.org/pdf/2106.02347.pdf)  

* **Co-evolution Transformer for Protein Contact Prediction**, *NeurIPS*, 2021. [\[pdf\]](https://proceedings.neurips.cc/paper_files/paper/2021/file/770f8e448d07586afbf77bb59f698587-Paper.pdf) [\[code\]](https://github.com/microsoft/ProteinFolding/tree/main/coevolution_transformer) 

* **Amortized Tree Generation for Bottom-up Synthesis Planning and Synthesizable Molecular Design**, *ICLR*, 2022. [\[pdf\]](https://arxiv.org/pdf/2110.06389.pdf) [\[code\]](https://github.com/wenhao-gao/SynNet) 

* **Iterative Refinement Graph Neural Network for Antibody Sequence-Structure Co-design**, *ICLR*, 2022. [\[pdf\]](https://arxiv.org/pdf/2110.04624.pdf)  

* **Ab-Initio Potential Energy Surfaces by Pairing GNNs with Neural Wave Functions**, *ICLR*, 2022. [\[pdf\]](https://arxiv.org/pdf/2110.05064.pdf)  

* **Independent SE(3)-Equivariant Models for End-to-End Rigid Protein Docking**, *ICLR*, 2022. [\[pdf\]](https://arxiv.org/pdf/2111.07786.pdf) [\[code\]](https://github.com/octavian-ganea/equidock_public) 

* **Geometric and Physical Quantities Improve E(3) Equivariant Message Passing**, *ICLR*, 2022. [\[pdf\]](https://arxiv.org/pdf/2110.02905.pdf)  

* **Chemical-Reaction-Aware Molecule Representation Learning**, *ICLR*, 2022. [\[pdf\]](https://arxiv.org/pdf/2109.09888.pdf) [\[code\]](https://github.com/hwwang55/MolR) 

* **OntoProtein: Protein Pretraining With Gene Ontology Embedding**, *ICLR*, 2022. [\[pdf\]](https://arxiv.org/pdf/2201.11147.pdf) [\[code\]](https://github.com/zjunlp/OntoProtein) 

* **Simple GNN Regularisation for 3D Molecular Property Prediction & Beyond**, *ICLR*, 2022. [\[pdf\]](https://arxiv.org/pdf/2106.07971.pdf) [\[code\]](https://github.com/deepmind/deepmind-research/tree/master/ogb_lsc/pcq) 

* **Spherical Message Passing for 3D Molecular Graphs**, *ICLR*, 2022. [\[pdf\]](https://openreview.net/pdf?id=givsRXsOt9r) [\[code\]](https://github.com/divelab/DIG) 

* **Crystal Diffusion Variational Autoencoder for Periodic Material Generation**, *ICLR*, 2022. [\[pdf\]](https://arxiv.org/pdf/2110.06197.pdf) [\[code\]](https://github.com/txie-93/cdvae) 

* **Spatial Graph Attention and Curiosity-driven Policy for Antiviral Drug Discovery**, *ICLR*, 2022. [\[pdf\]](https://arxiv.org/pdf/2106.02190.pdf) [\[code\]](https://github.com/yulun-rayn/DGAPN) 

* **Ancestral Protein Sequence Reconstruction using A Tree-structured Ornstein-Uhlenbeck Variational Autoencoder**, *ICLR*, 2022. [\[pdf\]](https://openreview.net/pdf?id=FZoZ7a31GCW) [\[code\]](https://github.com/LysSanzMoreta/DRAUPNIR_ASR) 

* **Differentiable Scaffolding Tree for Molecular Optimization**, *ICLR*, 2022. [\[pdf\]](https://openreview.net/pdf?id=w_drCosT76) [\[code\]](https://github.com/futianfan/DST) 

* **Pre-training Molecular Graph Representation with 3D Geometry**, *ICLR*, 2022. [\[pdf\]](https://arxiv.org/pdf/2110.07728.pdf) [\[code\]](https://github.com/chao1224/GraphMVP) 

* **An Autoregressive Flow Model for 3D Molecular Geometry Generation from Scratch**, *ICLR*, 2022. [\[pdf\]](https://openreview.net/pdf?id=C03Ajc-NS5W)  

* **Learning 3D Representations of Molecular Chirality with Invariance to Bond Rotations**, *ICLR*, 2022. [\[pdf\]](https://arxiv.org/pdf/2110.04383.pdf) [\[code\]](https://github.com/keiradams/ChIRo) 

* **Energy-Inspired Molecular Conformation Optimization**, *ICLR*, 2022. [\[pdf\]](https://openreview.net/pdf?id=7QfLW-XZTl) [\[code\]](https://github.com/guanjq/confopt_official) 

* **Learning to Extend Molecular Scaffolds with Structural Motifs**, *ICLR*, 2022. [\[pdf\]](https://arxiv.org/pdf/2103.03864.pdf) [\[code\]](https://github.com/microsoft/molecule-generation) 

* **A Benchmark for Experimental Design in Drug Discovery**, *ICLR*, 2022. [\[pdf\]](https://openreview.net/pdf?id=-w2oomO6qgc)  

* **Equivariant Graph Mechanics Networks with Constraints**, *ICLR*, 2022. [\[pdf\]](https://arxiv.org/pdf/2203.06442.pdf) [\[code\]](https://github.com/hanjq17/GMN) 

* **GeoDiff: a Geometric Diffusion Model for Molecular Conformation Generation**, *ICLR*, 2022. [\[pdf\]](https://arxiv.org/pdf/2203.02923.pdf) [\[code\]](https://github.com/MinkaiXu/GeoDiff)  

* **Geometric Transformers for Protein Interface Contact Prediction**, *ICLR*, 2022. [\[pdf\]](https://arxiv.org/pdf/2110.02423.pdf) [\[code\]](https://github.com/BioinfoMachineLearning/DeepInteract) 

* **Data-Efficient Graph Grammar Learning for Molecular Generation**, *ICLR*, 2022. [\[pdf\]](https://arxiv.org/pdf/2203.08031.pdf) [\[code\]](https://github.com/gmh14/data_efficient_grammar) 

* **TorchMD-NET: Equivariant Transformers for Neural Network based Molecular Potentials**, *ICLR*, 2022. [\[pdf\]](https://arxiv.org/pdf/2202.02541.pdf) [\[code\]](https://github.com/torchmd/torchmd-net) 

* **Spanning Tree-based Graph Generation for Molecules**, *ICLR*, 2022. [\[pdf\]](https://openreview.net/pdf?id=w60btE_8T2m)  

* **Generative Coarse-Graining of Molecular Conformations**, *ICML*, 2022. [\[pdf\]](https://arxiv.org/pdf/2201.12176.pdf) [\[code\]](https://github.com/wwang2/CoarseGrainingVAE)   

* **Retroformer: Pushing the Limits of End-to-end Retrosynthesis Transformer**, *ICML*, 2022. [\[pdf\]](https://proceedings.mlr.press/v162/wan22a/wan22a.pdf) [\[code\]](https://github.com/yuewan2/Retroformer) 

* **EQUIBIND: Geometric Deep Learning for Drug Binding Structure Prediction**, *ICML*, 2022. [\[pdf\]](https://arxiv.org/pdf/2202.05146.pdf) [\[code\]](https://github.com/HannesStark/EquiBind) 

* **3D Infomax improves GNNs for Molecular Property Prediction**, *ICML*, 2022. [\[pdf\]](https://arxiv.org/pdf/2110.04126.pdf) [\[code\]](https://github.com/HannesStark/3DInfomax) 

* **Accelerating Bayesian Optimization for Biological Sequence Design with Denoising Autoencoders**, *ICML*, 2022. [\[pdf\]](https://arxiv.org/pdf/2203.12742.pdf) [\[code\]](https://github.com/samuelstanton/lambo) 

* **Proximal Exploration for Model-guided Protein Sequence Design**, *ICML*, 2022. [\[pdf\]](https://proceedings.mlr.press/v162/ren22a/ren22a.pdf) [\[code\]](https://github.com/HeliXonProtein/proximal-exploration) 

* **Pocket2Mol: Efficient Molecular Sampling Based on 3D Protein Pockets**, *ICML*, 2022. [\[pdf\]](https://arxiv.org/pdf/2205.07249.pdf) [\[code\]](https://github.com/pengxingang/Pocket2Mol) 

* **Tranception: Protein Fitness Prediction with Autoregressive Transformers and Inference-time Retrieval**, *ICML*, 2022. [\[pdf\]](https://arxiv.org/pdf/2205.13760.pdf) [\[code\]](https://github.com/OATML-Markslab/Tranception) 

* **Path-Aware and Structure-Preserving Generation of Synthetically Accessible Molecules**, *ICML*, 2022. [\[pdf\]](https://proceedings.mlr.press/v162/noh22a/noh22a.pdf)  

* **Generating 3D Molecules for Target Protein Binding**, *ICML*, 2022. [\[pdf\]](https://arxiv.org/pdf/2204.09410.pdf)  

* **Query-Efficient and Scalable Black-Box Adversarial Attacks on Discrete Sequential Data via Bayesian Optimization**, *ICML*, 2022. [\[pdf\]](https://arxiv.org/pdf/2206.08575.pdf) [\[code\]](https://github.com/snu-mllab/DiscreteBlockBayesAttack) 

* **Score-based Generative Modeling of Graphs via the System of Stochastic Differential Equations**, *ICML*, 2022. [\[pdf\]](https://arxiv.org/pdf/2202.02514.pdf) [\[code\]](https://github.com/harryjo97/GDSS) 

* **Antibody-Antigen Docking and Design via Hierarchical Structure Refinement**, *ICML*, 2022. [\[pdf\]](https://proceedings.mlr.press/v162/jin22a/jin22a.pdf) [\[code\]](github.com/wengong-jin/abdockgen) 

* **Biological Sequence Design with GFlowNets**, *ICML*, 2022. [\[pdf\]](https://arxiv.org/pdf/2203.04115.pdf) [\[code\]](https://github.com/MJ10/BioSeq-GFN-AL) 

* **3DLinker: An E(3) Equivariant Variational Autoencoder for Molecular Linker Design**, *ICML*, 2022. [\[pdf\]](https://arxiv.org/pdf/2205.07309.pdf)  

* **Learning Inverse Folding from Millions of Predicted Structures**, *ICML*, 2022. [\[pdf\]](https://proceedings.mlr.press/v162/hsu22a/hsu22a.pdf)  

* **Equivariant Diffusion for Molecule Generation in 3D**, *ICML*, 2022. [\[pdf\]](https://arxiv.org/pdf/2203.17003.pdf)  

* **Deep Squared Euclidean Approximation to the Levenshtein Distance for DNA Storage**, *ICML*, 2022. [\[pdf\]](https://arxiv.org/pdf/2207.04684.pdf)  

* **How to Fill the Optimum Set? Population Gradient Descent with Harmless Diversity**, *ICML*, 2022. [\[pdf\]](https://proceedings.mlr.press/v162/gong22b/gong22b.pdf)  

* **Conformal Prediction Sets with Limited False Positives**, *ICML*, 2022. [\[pdf\]](https://proceedings.mlr.press/v162/fisch22a/fisch22a.pdf) [\[code\]](https://github.com/ajfisch/conformal-fp) 

* **LIMO: Latent Inceptionism for Targeted Molecule Generation**, *ICML*, 2022. [\[pdf\]](https://arxiv.org/pdf/2206.09010.pdf) [\[code\]](https://github.com/Rose-STL-Lab/LIMO) 

* **SE(3) Equivariant Graph Neural Networks with Complete Local Frames**, *ICML*, 2022. [\[pdf\]](https://arxiv.org/pdf/2110.14811.pdf) [\[code\]]() 

* **One-Pass Diversified Sampling with Application to Terabyte-Scale Genomic Sequence Streams**, *ICML*, 2022. [\[pdf\]](https://proceedings.mlr.press/v162/coleman22a/coleman22a.pdf)  

* **Molecular Representation Learning via Heterogeneous Motif Graph Neural Networks**, *ICML*, 2022. [\[pdf\]](https://arxiv.org/pdf/2202.00529.pdf) [\[code\]](https://github.com/ZhaoningYu1996/HM-GNN) 

* **Periodic Graph Transformers for Crystal Material Property Prediction**, *NeurIPS*, 2022. [\[pdf\]](https://arxiv.org/pdf/2209.11807.pdf) 
 
* **Amortized Inference for Heterogeneous Reconstruction in Cryo-EM**, *NeurIPS*, 2022. [\[pdf\]](https://arxiv.org/pdf/2210.07387.pdf)  

* **Modular Flows: Differential Molecular Generation**, *NeurIPS*, 2022. [\[pdf\]](https://arxiv.org/pdf/2210.06032.pdf)  

* **Sparse Fourier Backpropagation in Cryo-EM Reconstruction**, *NeurIPS*, 2022. [\[pdf\]](https://openreview.net/pdf?id=51f5sPXJD_E) [\[code\]](https://github.com/dkimanius/sbackprop) 

* **Reinforced Genetic Algorithm for Structure-based Drug Design**, *NeurIPS*, 2022. [\[pdf\]](https://arxiv.org/pdf/2211.16508.pdf) [\[code\]](https://github.com/futianfan/reinforced-genetic-algorithm)  

* **MACE: Higher Order Equivariant Message Passing Neural Networks for Fast and Accurate Force Fields**, *NeurIPS*, 2022. [\[pdf\]](https://arxiv.org/pdf/2206.07697.pdf) [\[code\]](https://github.com/ACEsuit/mace) 

* **Gold-standard Solutions to the Schrödinger Equation using Deep Learning: How Much Physics Do We Need?**, *NeurIPS*, 2022. [\[pdf\]](https://arxiv.org/pdf/2205.09438.pdf) [\[code\]](https://github.com/mdsunivie/deeperwin) 

* **GRASP: Navigating Retrosynthetic Planning with Goal-driven Policy**, *NeurIPS*, 2022. [\[pdf\]](https://openreview.net/pdf?id=ofwkaIWFqqv)  

* **Regularized Molecular Conformation Fields**, *NeurIPS*, 2022. [\[pdf\]](https://openreview.net/pdf?id=7XCFxnG8nGS) [\[code\]](https://github.com/leowang1217/RMCF) 

* **Cross-Linked Unified Embedding for Cross-modality Representation Learning**, *NeurIPS*, 2022. [\[pdf\]](https://openreview.net/pdf?id=Tfb73TeKnJ-) [\[code\]](https://github.com/gao-lab/GLUE) 

* **Antigen-Specific Antibody Design and Optimization with Diffusion-Based Generative Models for Protein Structures**, *NeurIPS*, 2022. [\[pdf\]](https://openreview.net/pdf?id=jSorGn2Tjg) [\[code\]](https://github.com/luost26/diffab) 

* **Equivariant Graph Hierarchy-Based Neural Networks**, *NeurIPS*, 2022. [\[pdf\]](https://arxiv.org/pdf/2202.10643.pdf) [\[code\]](https://github.com/hanjq17/EGHN) 

* **An Efficient Graph Generative Model for Navigating Ultra-large Combinatorial Synthesis Libraries**, *NeurIPS*, 2022. [\[pdf\]](https://arxiv.org/pdf/2211.04468.pdf) [\[code\]](https://github.com/AtomwiseInc/cslvae) 

* **Residual Multiplicative Filter Networks for Multiscale Reconstruction**, *NeurIPS*, 2022. [\[pdf\]](https://arxiv.org/pdf/2206.00746.pdf) [\[code\]](https://shekshaa.github.io/ResidualMFN/) 

* **TANKBind: Trigonometry-Aware Neural NetworKs for Drug-Protein Binding Structure Prediction**, *NeurIPS*, 2022. [\[pdf\]](https://openreview.net/pdf?id=MSBDFwGYwwt) [\[code\]]() 

* **ComENet: Towards Complete and Efficient Message Passing for 3D Molecular Graphs**, *NeurIPS*, 2022. [\[pdf\]](https://arxiv.org/pdf/2206.08515.pdf)   

* **On the Symmetries of the Synchronization Problem in Cryo-EM: Multi-Frequency Vector Diffusion Maps on the Projective Plane**, *NeurIPS*, 2022. [\[pdf\]](https://openreview.net/pdf?id=owDcdLGgEm)  

* **Equivariant Networks for Crystal Structures**, *NeurIPS*, 2022. [\[pdf\]](https://arxiv.org/pdf/2211.15420.pdf)  

* **Exploring Evolution-aware & -free Protein Language Models as Protein Function Predictors**, *NeurIPS*, 2022. [\[pdf\]](https://arxiv.org/pdf/2206.06583.pdf) [\[code\]](https://github.com/elttaes/Revisiting-PLMs) 

* **Diffusion-based Molecule Generation with Informative Prior Bridges**, *NeurIPS*, 2022. [\[pdf\]](https://arxiv.org/pdf/2209.00865.pdf)  

* **Local Latent Space Bayesian Optimization over Structured Inputs**, *NeurIPS*, 2022. [\[pdf\]](https://arxiv.org/pdf/2201.11872.pdf) [\[code\]](https://github.com/nataliemaus/lolbo) 

* **Unsupervised Learning of Group Invariant and Equivariant Representations**, *NeurIPS*, 2022. [\[pdf\]](https://arxiv.org/pdf/2202.07559.pdf)  

* **Symmetry-induced Disentanglement on Graphs**, *NeurIPS*, 2022. [\[pdf\]](https://openreview.net/pdf?id=4tM0P_4N8D9)  

* **So3krates: Equivariant Attention for Interactions on Arbitrary Length-scales in Molecular Systems**, *NeurIPS*, 2022. [\[pdf\]](https://arxiv.org/pdf/2205.14276.pdf) [\[code\]](https://github.com/thorben-frank/mlff) 

* **Multi-objective Deep Data Generation with Correlated Property Control**, *NeurIPS*, 2022. [\[pdf\]](https://arxiv.org/pdf/2210.01796.pdf) [\[code\]](https://github.com/shi-yu-wang/CorrVAE) 

* **ToDD: Topological Compound Fingerprinting in Computer-Aided Drug Discovery**, *NeurIPS*, 2022. [\[pdf\]](https://arxiv.org/pdf/2211.03808.pdf)  

* **Debiased, Longitudinal and Coordinated Drug Recommendation through Multi-Visit Clinic Records**, *NeurIPS*, 2022. [\[pdf\]](https://openreview.net/pdf?id=zVglD2W0EAS) [\[code\]](https://github.com/ssshddd/DrugRec) 

* **Probabilistic Transformer: Modelling Ambiguities and Distributions for RNA Folding and Molecule Design**, *NeurIPS*, 2022. [\[pdf\]](https://arxiv.org/pdf/2205.13927.pdf) [\[code\]](https://github.com/automl/ProbTransformer) 

* **Predicting Cellular Responses to Novel Drug Perturbations at a Single-Cell Resolution**, *NeurIPS*, 2022. [\[pdf\]](https://arxiv.org/pdf/2204.13545.pdf) [\[code\]](https://github.com/theislab/chemCPA) 

* **Molecule Generation by Principal Subgraph Mining and Assembling**, *NeurIPS*, 2022. [\[pdf\]](https://arxiv.org/pdf/2106.15098.pdf) [\[code\]](https://github.com/THUNLP-MT/PS-VAE) 

* **Torsional Diffusion for Molecular Conformer Generation**, *NeurIPS*, 2022. [\[pdf\]](https://arxiv.org/pdf/2206.01729.pdf) [\[code\]](https://github.com/gcorso/torsional-diffusion) 

* **Zero-Shot 3D Drug Design by Sketching and Generating**, *NeurIPS*, 2022. [\[pdf\]](https://arxiv.org/pdf/2209.13865.pdf) [\[code\]](https://github.com/longlongman/DESERT) 

* **Protein Sequence and Structure Co-Design with Equivariant Translation**, *ICLR*, 2023. [\[pdf\]](https://arxiv.org/pdf/2210.08761.pdf) [\[code\]](https://github.com/sokrypton/ColabDesign)

* **D4FT: A Deep Learning Approach to Kohn-Sham Density Functional Theory**, *ICLR*, 2023. [\[pdf\]](https://arxiv.org/pdf/2303.00399.pdf) [\[code\]](https://github.com/sail-sg/d4ft)

* **Anisotropic Message Passing: Graph Neural Networks with Directional and Long-Range Interactions**, *ICLR*, 2023. [\[pdf\]](https://openreview.net/pdf?id=socffUzSIlx) 

* **Protein Representation Learning via Knowledge Enhanced Primary Structure Modeling**, *ICLR*, 2023. [\[pdf\]](https://arxiv.org/pdf/2301.13154.pdf) [\[code\]](https://github.com/RL4M/KeAP)

* **Learning Harmonic Molecular Representations on Riemannian Manifold**, *ICLR*, 2023. [\[pdf\]](https://arxiv.org/pdf/2303.15520.pdf) [\[code\]](https://github.com/GeomMolDesign/HMR)

* **PiFold: Toward Effective and Efficient Protein Inverse Folding**, *ICLR*, 2023. [\[pdf\]](https://arxiv.org/pdf/2209.12643.pdf) [\[code\]](https://github.com/A4Bio/PiFold)

* **A Self-Attention Ansatz for Ab-initio Quantum Chemistry**, *ICLR*, 2023. [\[pdf\]](https://openreview.net/pdf?id=xveTeHVlF7j) [\[code\]](https://github.com/google-deepmind/ferminet)

* **Diffusion Probabilistic Modeling of Protein Backbones in 3D for the Motif-scaffolding Problem**, *ICLR*, 2023. [\[pdf\]](https://arxiv.org/pdf/2206.04119.pdf) [\[code\]](https://github.com/blt2114/ProtDiff_SMCDiff)

* **DiGress: Discrete Denoising Diffusion for Graph Generation**, *ICLR*, 2023. [\[pdf\]](https://arxiv.org/pdf/2209.14734.pdf) [\[code\]](https://github.com/cvignac/DiGress)

* **Matching Receptor to Odorant with Protein Language and Graph Neural Networks**, *ICLR*, 2023. [\[pdf\]](https://openreview.net/pdf?id=q9VherQJd8_) [\[code\]](https://github.com/MatejHl/Receptor2Odorant)

* **Sampling-free Inference for Ab-Initio Potential Energy Surface Networks**, *ICLR*, 2023. [\[pdf\]](https://arxiv.org/pdf/2205.14962.pdf) [\[code\]](https://github.com/n-gao/pesnet)

* **E3Bind: An End-to-End Equivariant Network for Protein-Ligand Docking**, *ICLR*, 2023. [\[pdf\]](https://arxiv.org/pdf/2210.06069.pdf) 

* **Retrieval-based Controllable Molecule Generation**, *ICLR*, 2023. [\[pdf\]](https://arxiv.org/pdf/2208.11126.pdf) [\[code\]](https://github.com/NVlabs/RetMol)

* **How Much Space Has Been Explored? Measuring the Chemical Space Covered by Databases and Machine-Generated Molecules**, *ICLR*, 2023. [\[pdf\]](https://arxiv.org/pdf/2112.12542.pdf) [\[code\]](https://github.com/yutxie/chem-measure)

* **Protein Representation Learning by Geometric Structure Pretraining**, *ICLR*, 2023. [\[pdf\]](https://arxiv.org/pdf/2203.06125.pdf) [\[code\]](https://github.com/DeepGraphLearning/GearNet)

* **Generative Augmented Flow Networks**, *ICLR*, 2023. [\[pdf\]](https://openreview.net/pdf?id=urF_CBK5XC0) [\[code\]](https://github.com/ling-pan/GAFN)

* **Rotamer Density Estimator is an Unsupervised Learner of the Effect of Mutations on Protein-Protein Interaction**, *ICLR*, 2023. [\[pdf\]](https://openreview.net/pdf?id=_X9Yl1K2mD) [\[code\]](https://github.com/luost26/RDE-PPI) 

* **Relational Attention: Generalizing Transformers for Graph-Structured Tasks**, *ICLR*, 2023. [\[pdf\]](https://arxiv.org/pdf/2210.05062.pdf) [\[code\]](https://github.com/CameronDiao/relational-transformer)

* **3D Equivariant Diffusion for Target-Aware Molecule Generation and Affinity Prediction**, *ICLR*, 2023. [\[pdf\]](https://arxiv.org/pdf/2303.03543.pdf) [\[code\]](https://github.com/guanjq/targetdiff)

* **De Novo Molecular Generation via Connection-aware Motif Mining**, *ICLR*, 2023. [\[pdf\]](https://arxiv.org/pdf/2302.01129.pdf) [\[code\]](https://github.com/MIRALab-USTC/AI4Sci-MiCaM)

* **Uni-Mol: A Universal 3D Molecular Representation Learning Framework**, *ICLR*, 2023. [\[pdf\]](https://openreview.net/pdf?id=6K2RM6wVqKu) [\[code\]](https://github.com/dptech-corp/Uni-Mol)

* **Multi-level Protein Structure Pre-training via Prompt Learning**, *ICLR*, 2023. [\[pdf\]](https://openreview.net/pdf?id=XGagtiJ8XC) [\[code\]](https://github.com/HICAI-ZJU/PromptProtein)

* **Context-enriched Molecule Representations Improve Few-shot Drug Discovery**, *ICLR*, 2023. [\[pdf\]](https://arxiv.org/pdf/2305.09481.pdf) [\[code\]](https://github.com/ml-jku/MHNfs)

* **Equivariant Energy-Guided SDE for Inverse Molecular Design**, *ICLR*, 2023. [\[pdf\]](https://arxiv.org/pdf/2209.15408.pdf) [\[code\]](https://github.com/gracezhao1997/EEGSDE)

* **Pre-training via Denoising for Molecular Property Prediction**, *ICLR*, 2023. [\[pdf\]](https://openreview.net/pdf?id=tYIMtogyee) [\[code\]](https://github.com/shehzaidi/pre-training-via-denoising)

* **Sparsity May Cry: Let Us Fail (Current) Sparse Neural Networks Together!**, *ICLR*, 2023. [\[pdf\]](https://arxiv.org/pdf/2303.02141.pdf) [\[code\]](https://github.com/VITA-Group/SMC-Bench)

* **$\mathcal{O}$-GNN: Incorporating Ring Priors into Molecular Modeling**, *ICLR*, 2023. [\[pdf\]](https://openreview.net/pdf?id=5cFfz6yMVPU) [\[code\]](https://github.com/O-GNN/O-GNN)

* **Multiple Sequence Alignment as a Sequence-to-sequence Learning Problem**, *ICLR*, 2023. [\[pdf\]](https://openreview.net/pdf?id=8efJYMBrNb) 

* **One Transformer Can Understand Both 2D & 3D Molecular Data**, *ICLR*, 2023. [\[pdf\]](https://arxiv.org/pdf/2210.01765.pdf) [\[code\]](https://github.com/lsj2408/Transformer-M)

* **A Graph Neural Network Approach to Automated Model Building in Cryo-EM Maps**, *ICLR*, 2023. [\[pdf\]](https://arxiv.org/pdf/2210.00006.pdf) [\[code\]](https://github.com/3dem/model-angelo)

* **Continuous-Discrete Convolution for Geometry-Sequence Modeling in Proteins**, *ICLR*, 2023. [\[pdf\]](https://openreview.net/pdf?id=P5Z-Zl9XJ7) [\[code\]](https://github.com/hehefan/Continuous-Discrete-Convolution)

* **Molecular Geometry Pretraining with SE(3)-Invariant Denoising Distance Matching**, *ICLR*, 2023. [\[pdf\]](https://arxiv.org/pdf/2206.13602.pdf) [\[code\]](https://github.com/chao1224/GeoSSL)

* **Equiformer: Equivariant Graph Attention Transformer for 3D Atomistic Graphs**, *ICLR*, 2023. [\[pdf\]](https://arxiv.org/pdf/2206.11990.pdf) [\[code\]](https://github.com/atomicarchitects/equiformer)

* **Meta-learning Adaptive Deep Kernel Gaussian Processes for Molecular Property Prediction**, *ICLR*, 2023. [\[pdf\]](https://arxiv.org/pdf/2205.02708.pdf) [\[code\]](https://github.com/Wenlin-Chen/ADKF-IFT)

* **Molecule Generation For Target Protein Binding with Structural Motifs**, *ICLR*, 2023. [\[pdf\]](https://openreview.net/pdf?id=Rq13idF0F73) [\[code\]](https://github.com/zaixizhang/FLAG)

* **On Pre-trained Language Models for Antibody**, *ICLR*, 2023. [\[pdf\]](https://arxiv.org/pdf/2301.12112.pdf) [\[code\]](https://github.com/dqwang122/EATLM)

* **Unsupervised Manifold Alignment with Joint Multidimensional Scaling**, *ICLR*, 2023. [\[pdf\]](https://arxiv.org/pdf/2207.02968.pdf) [\[code\]](https://github.com/BorgwardtLab/JointMDS)

* **Equivariant Shape-Conditioned Generation of 3D Molecules for Ligand-Based Drug Design**, *ICLR*, 2023. [\[pdf\]](https://openreview.net/pdf?id=4MbGnp4iPQ) [\[code\]](https://github.com/keiradams/SQUID)

* **Learning Hierarchical Protein Representations via Complete 3D Graph Networks**, *ICLR*, 2023. [\[pdf\]](https://arxiv.org/pdf/2207.12600.pdf) [\[code\]](https://github.com/divelab/DIG)

* **DiffDock: Diffusion Steps, Twists, and Turns for Molecular Docking**, *ICLR*, 2023. [\[pdf\]](https://arxiv.org/pdf/2210.01776.pdf) [\[code\]](https://github.com/gcorso/DiffDock)

* **Conditional Antibody Design as 3D Equivariant Graph Translation**, *ICLR*, 2023. [\[pdf\]](https://arxiv.org/pdf/2208.06073.pdf) [\[code\]](https://github.com/THUNLP-MT/MEAN)

* **MolDiff: Addressing the Atom-Bond Inconsistency Problem in 3D Molecule Diffusion Generation**, *ICML*, 2023. [\[pdf\]](https://arxiv.org/pdf/2305.07508.pdf) [\[code\]](https://github.com/pengxingang/MolDiff)  

* **Retrosynthetic Planning with Dual Value Networks**, *ICML*, 2023. [\[pdf\]](https://arxiv.org/pdf/2301.13755.pdf) 

* **FusionRetro: Molecule Representation Fusion via In-Context Learning for Retrosynthetic Planning**, *ICML*, 2023. [\[pdf\]](https://arxiv.org/pdf/2209.15315.pdf) [\[code\]](https://github.com/SongtaoLiu0823/FusionRetro)

* **A Group Symmetric Stochastic Differential Equation Model for Molecule Multi-modal Pretraining**, *ICML*, 2023. [\[pdf\]](https://arxiv.org/pdf/2305.18407.pdf) [\[code\]](https://github.com/chao1224/MoleculeSDE)

* **Efficient Approximations of Complete Interatomic Potentials for Crystal Property Prediction**, *ICML*, 2023. [\[pdf\]](https://arxiv.org/pdf/2306.10045.pdf) [\[code\]](https://github.com/divelab/AIRS/tree/main/OpenMat/PotNet) 

* **Generating Novel, Designable, and Diverse Protein Structures by Equivariantly Diffusing Oriented Residue Clouds**, *ICML*, 2023. [\[pdf\]](https://arxiv.org/pdf/2301.12485.pdf) [\[code\]](https://github.com/aqlaboratory/genie)

* **Exploring Chemical Space with Score-based Out-of-distribution Generation**, *ICML*, 2023. [\[pdf\]](https://arxiv.org/pdf/2206.07632.pdf) [\[code\]](https://github.com/SeulLee05/MOOD)

* **Conditional Graph Information Bottleneck for Molecular Relational Learning**, *ICML*, 2023. [\[pdf\]](https://arxiv.org/pdf/2305.01520.pdf) [\[code\]](https://github.com/Namkyeong/CGIB)

* **Ewald-based Long-Range Message Passing for Molecular Graphs**, *ICML*, 2023. [\[pdf\]](https://arxiv.org/pdf/2303.04791.pdf) [\[code\]](https://github.com/arthurkosmala/EwaldMP)

* **End-to-End Full-Atom Antibody Design**, *ICML*, 2023. [\[pdf\]](https://arxiv.org/pdf/2302.00203.pdf) [\[code\]](https://github.com/THUNLP-MT/dyMEAN) 

* **Autoregressive Diffusion Model for Graph Generation**, *ICML*, 2023. [\[pdf\]](https://arxiv.org/pdf/2307.08849.pdf) [\[code\]](https://github.com/caio-freitas/GraphARM)

* **Rigid Body Flows for Sampling Molecular Crystal Structures**, *ICML*, 2023. [\[pdf\]](https://arxiv.org/pdf/2301.11355.pdf) [\[code\]](https://github.com/noegroup/rigid-flows)

* **Drug Discovery under Covariate Shift with Domain-Informed Prior Distributions over Functions**, *ICML*, 2023. [\[pdf\]](https://arxiv.org/pdf/2307.15073.pdf) [\[code\]](https://github.com/leojklarner/Q-SAVI)

* **SurProGenes: Survival Risk-Ordered Representation of Cancer Patients and Genes for the Identification of Prognostic Genes**, *ICML*, 2023. [\[pdf\]](https://proceedings.mlr.press/v202/kim23s/kim23s.pdf) [\[code\]](https://github.com/JunetaeKim/SurProGenes)

* **Biases in Evaluation of Molecular Optimization Methods and Bias Reduction Strategies**, *ICML*, 2023. [\[pdf\]](https://proceedings.mlr.press/v202/kajino23a/kajino23a.pdf) [\[code\]](https://github.com/kanojikajino/biases-in-mol-opt) 

* **On the Expressive Power of Geometric Graph Neural Networks**, *ICML*, 2023. [\[pdf\]](https://arxiv.org/pdf/2301.09308.pdf) [\[code\]](https://github.com/chaitjo/geometric-gnn-dojo)

* **Multi-Objective GFlowNets**, *ICML*, 2023. [\[pdf\]](https://arxiv.org/pdf/2210.12765.pdf) [\[code\]](https://github.com/GFNOrg/multi-objective-gfn)

* **Hierarchical Grammar-Induced Geometry for Data-Efficient Molecular Property Prediction**, *ICML*, 2023. [\[pdf\]](https://proceedings.mlr.press/v202/guo23h/guo23h.pdf) [\[code\]](https://github.com/gmh14/Geo-DEG)

* **DECOMPDIFF: Diffusion Models with Decomposed Priors for Structure-Based Drug Design**, *ICML*, 2023. [\[pdf\]](https://proceedings.mlr.press/v202/guan23a/guan23a.pdf) [\[code\]](https://github.com/bytedance/DecompDiff)

* **Generalizing Neural Wave Functions**, *ICML*, 2023. [\[pdf\]](https://arxiv.org/pdf/2302.04168.pdf) [\[code\]](https://github.com/n-gao/globe) 

* **Fractional Denoising for 3D Molecular Pre-training**, *ICML*, 2023. [\[pdf\]](https://arxiv.org/pdf/2307.10683.pdf) [\[code\]](https://github.com/fengshikun/Frad)

* **Scaling Spherical CNNs**, *ICML*, 2023. [\[pdf\]](https://arxiv.org/pdf/2306.05420.pdf) [\[code\]](https://github.com/google-research/spherical-cnn)

* **E(n) Equivariant Message Passing Simplicial Networks**, *ICML*, 2023. [\[pdf\]](https://arxiv.org/pdf/2305.07100.pdf) 

* **FAENet: Frame Averaging Equivariant GNN for Materials Modeling**, *ICML*, 2023. [\[pdf\]](https://arxiv.org/pdf/2305.05577.pdf) [\[code\]](https://faenet.readthedocs.io/en/latest/)

* **A Kernelized Stein Discrepancy for Biological Sequences**, *ICML*, 2023. [\[pdf\]](https://proceedings.mlr.press/v202/amin23a/amin23a.pdf)  

* **Improving Graph Generation by Restricting Graph Bandwidth**, *ICML*, 2023. [\[pdf\]](https://arxiv.org/pdf/2301.10857.pdf) [\[code\]](https://github.com/Genentech/bandwidth-graph-generation)

* **Unifying Molecular and Textual Representations via Multi-task Language Modelling**, *ICML*, 2023. [\[pdf\]](https://arxiv.org/pdf/2301.12586.pdf) [\[code\]](https://github.com/GT4SD/multitask_text_and_chemistry_t5)

* **Bidirectional Learning for Offline Model-based Biological Sequence Design**, *ICML*, 2023. [\[pdf\]](https://arxiv.org/pdf/2301.02931.pdf) [\[code\]](https://github.com/GGchen1997/BIB-ICML2023-Submission)

* **Structure-informed Language Models Are Protein Designers**, *ICML*, 2023. [\[pdf\]](https://arxiv.org/pdf/2302.01649.pdf) [\[code\]](https://github.com/BytedProtein/ByProt)

* **Learning Subpocket Prototypes for Generalizable Structure-based Drug Design**, *ICML*, 2023. [\[pdf\]](https://arxiv.org/pdf/2305.13997.pdf) [\[code\]](https://github.com/zaixizhang/FLAG)

* **Efficient and Equivariant Graph Networks for Predicting Quantum Hamiltonian**, *ICML*, 2023. [\[pdf\]](https://arxiv.org/pdf/2306.04922.pdf) [\[code\]](https://github.com/divelab/AIRS)

* **SE(3) diffusion model with application to protein backbone generation**, *ICML*, 2023. [\[pdf\]](https://arxiv.org/pdf/2302.02277.pdf) [\[code\]](https://github.com/jasonkyuyim/se3_diffusion)

* **Chemically Transferable Generative Backmapping of Coarse-Grained Proteins**, *ICML*, 2023. [\[pdf\]](https://arxiv.org/pdf/2303.01569.pdf) [\[code\]](https://github.com/learningmatter-mit/GenZProt)

* **Quantum 3D Graph Learning with Applications to Molecule Embedding**, *ICML*, 2023. [\[pdf\]](https://proceedings.mlr.press/v202/yan23e/yan23e.pdf)  

* **ProtST: Multi-Modality Learning of Protein Sequences and Biomedical Texts**, *ICML*, 2023. [\[pdf\]](https://arxiv.org/pdf/2301.12040.pdf) [\[code\]](https://github.com/DeepGraphLearning/ProtST)

* **Geometric Latent Diffusion Models for 3D Molecule Generation**, *ICML*, 2023. [\[pdf\]](https://arxiv.org/pdf/2305.01140.pdf) [\[code\]](https://github.com/MinkaiXu/GeoLDM)

* **QuantumDARTS: Differentiable Quantum Architecture Search for Variational Quantum Algorithms**, *ICML*, 2023. [\[pdf\]](https://proceedings.mlr.press/v202/wu23v/wu23v.pdf) 

* **Uncertainty Estimation for Molecules: Desiderata and Methods**, *ICML*, 2023. [\[pdf\]](https://arxiv.org/pdf/2306.14916.pdf) [\[code\]](https://github.com/wollschl/uncertainty_for_molecules)

* **AbODE: Ab Initio Antibody Design using Conjoined ODEs**, *ICML*, 2023. [\[pdf\]](https://arxiv.org/pdf/2306.01005.pdf) [\[code\]](https://github.com/yogeshverma1998/AbODE_)

* **Von Mises Mixture Distributions for Molecular Conformation Generation**, *ICML*, 2023. [\[pdf\]](https://arxiv.org/pdf/2306.07472.pdf) [\[code\]](https://github.com/thejonaslab/vonmises-icml-2023) 

* **Importance Weighted Expectation-Maximization for Protein Sequence Design**, *ICML*, 2023. [\[pdf\]](https://arxiv.org/pdf/2305.00386.pdf) 

* **Machine Learning Force Fields with Data Cost Aware Training**, *ICML*, 2023. [\[pdf\]](https://proceedings.mlr.press/v202/bukharin23a/bukharin23a.pdf) [\[code\]](https://github.com/abukharin3/asteroid)

* **Enhancing Activity Prediction Models in Drug Discovery with the Ability to Understand Human Language**, *ICML*, 2023. [\[pdf\]](https://arxiv.org/pdf/2303.03363.pdf) [\[code\]](https://github.com/ml-jku/clamp)

* **Geometric Clifford Algebra Networks**, *ICML*, 2023. [\[pdf\]](https://arxiv.org/pdf/2302.06594.pdf) [\[code\]](https://github.com/microsoft/cliffordlayers)

* **Coarse-to-Fine: a Hierarchical Diffusion Model for Molecule Generation in 3D**, *ICML*, 2023. [\[pdf\]](https://arxiv.org/pdf/2305.13266.pdf) [\[code\]](https://github.com/qiangbo1222/HierDiff)

* **JAWS-X: Addressing Efficiency Bottlenecks of Conformal Prediction Under Standard and Feedback Covariate Shif**, *ICML*, 2023. [\[pdf\]](https://proceedings.mlr.press/v202/prinster23a/prinster23a.pdf) [\[code\]](https://github.com/drewprinster/jaws-x)

* **Reducing SO(3) Convolutions to SO(2) for Efficient Equivariant GNNs**, *ICML*, 2023. [\[pdf\]](https://arxiv.org/pdf/2302.03655.pdf) [\[code\]](https://github.com/lucidrains/equiformer-pytorch) 

* **Better Training of GFlowNets with Local Credit and Incomplete Trajectories**, *ICML*, 2023. [\[pdf\]](https://arxiv.org/pdf/2302.01687.pdf) [\[code\]](https://github.com/ling-pan/FL-GFN)

* **Extrapolative Controlled Sequence Generation via Iterative Refinement**, *ICML*, 2023. [\[pdf\]](https://arxiv.org/pdf/2303.04562.pdf) [\[code\]](https://github.com/vishakhpk/iter-extrapolation)

* **Efficiently Predicting High Resolution Mass Spectra with Graph Neural Networks**, *ICML*, 2023. [\[pdf\]](https://proceedings.mlr.press/v202/murphy23a/murphy23a.pdf) [\[code\]](https://github.com/murphy17/graff-ms)

* **STRIDERNET: A Graph Reinforcement Learning Approach to Optimize Atomic Structures on Rough Energy Landscapes**, *ICML*, 2023. [\[pdf\]](https://proceedings.mlr.press/v202/bihani23a/bihani23a.pdf) [\[code\]](https://github.com/M3RG-IITD/StriderNET)

* **Reprogramming Pretrained Language Models for Antibody Sequence Infilling**, *ICML*, 2023. [\[pdf\]](https://arxiv.org/pdf/2210.07144.pdf) [\[code\]](https://github.com/IBM/ReprogBERT)

* **Accelerating Molecular Graph Neural Networks via Knowledge Distillation**, *NeurIPS*, 2023. [\[pdf\]](https://arxiv.org/pdf/2306.14818.pdf) [\[code\]](https://github.com/gasteigerjo/ocp/blob/main/DISTILL.md)

* **LinkerNet: Fragment Poses and Linker Co-Design with 3D Equivariant Diffusion**, *NeurIPS*, 2023. [\[pdf\]](https://openreview.net/pdf?id=6EaLIw3W7c) [\[code\]](https://github.com/guanjq/LinkerNet)

* **AI for Interpretable Chemistry: Predicting Radical Mechanistic Pathways via Contrastive Learning**, *NeurIPS*, 2023. [\[pdf\]](https://openreview.net/pdf?id=6kRQTPEVip) 

* **TensorNet: Cartesian Tensor Representations for Efficient Learning of Molecular Potentials**, *NeurIPS*, 2023. [\[pdf\]](https://arxiv.org/pdf/2306.06482.pdf) [\[code\]](https://github.com/torchmd/torchmd-net)

* **Implicit Transfer Operator Learning: Multiple Time-Resolution Surrogates for Molecular Dynamics**, *NeurIPS*, 2023. [\[pdf\]](https://arxiv.org/pdf/2305.18046.pdf) [\[code\]](https://github.com/olsson-group/ito)

* **GeoTMI: Predicting Quantum Chemical Property with Easy-to-Obtain Geometry via Positional Denoising**, *NeurIPS*, 2023. [\[pdf\]](https://arxiv.org/pdf/2304.03724.pdf) 

* **De novo Drug Design using Reinforcement Learning with Multiple GPT Agents**, *NeurIPS*, 2023. [\[pdf\]](https://arxiv.org/pdf/2401.06155.pdf) [\[code\]](https://github.com/HXYfighter/MolRL-MGPT)

* **Unsupervised Protein-Ligand Binding Energy Prediction via Neural Euler’s Rotation Equation**, *NeurIPS*, 2023. [\[pdf\]](https://arxiv.org/pdf/2301.10814.pdf) [\[code\]](https://github.com/wengong-jin/DSMBind)

* **May the Force be with You: Unified Force-Centric Pre-Training for 3D Molecular Conformations**, *NeurIPS*, 2023. [\[pdf\]](https://openreview.net/pdf?id=Ge8Mhggq0z) 

* **Smooth, Exact Rotational Symmetrization for Deep Learning on Point Clouds**, *NeurIPS*, 2023. [\[pdf\]](https://arxiv.org/pdf/2305.19302.pdf) [\[code\]](https://github.com/serfg/pet)

* **AbDiffuser: Full-Atom Generation of in vitro Functioning Antibodies**, *NeurIPS*, 2023. [\[pdf\]](https://openreview.net/pdf?id=7GyYpomkEa) 

* **Predicting Mutational Effects on Protein-protein Binding via a Side-chain Diffusion Probabilistic Model**, *NeurIPS*, 2023. [\[pdf\]](https://arxiv.org/pdf/2310.19849.pdf) [\[code\]](https://github.com/EureKaZhu/DiffAffinity/)

* **Pre-Training Protein Encoder via Siamese Sequence-Structure Diffusion Trajectory Prediction**, *NeurIPS*, 2023. [\[pdf\]](https://openreview.net/pdf?id=tIzbNQko3c) [\[code\]](https://github.com/DeepGraphLearning/SiamDiff)

* **Fragment-based Pretraining and Finetuning on Molecular Graphs**, *NeurIPS*, 2023. [\[pdf\]](https://arxiv.org/pdf/2310.03274.pdf) [\[code\]](https://github.com/lvkd84/GraphFP)

* **Disentangled Wasserstein Autoencoder for T-Cell Receptor Engineering**, *NeurIPS*, 2023. [\[pdf\]](https://arxiv.org/pdf/2210.08171.pdf) 

* **Optimizing over Trained GNNs via Symmetry Breaking**, *NeurIPS*, 2023. [\[pdf\]](https://arxiv.org/pdf/2305.09420.pdf) [\[code\]](https://github.com/cog-imperial/GNN_MIP_CAMD)

* **Molecule Joint Auto-Encoding: Trajectory Pretraining with 2D and 3D Diffusion**, *NeurIPS*, 2023. [\[pdf\]](https://arxiv.org/pdf/2312.03475.pdf) 

* **Generating QM1B with PySCFIPU**, *NeurIPS*, 2023. [\[pdf\]](https://arxiv.org/pdf/2311.01135.pdf) [\[code\]](https://github.com/graphcore-research/pyscf-ipu)

* **Compositional Sculpting of Iterative Generative Processes**, *NeurIPS*, 2023. [\[pdf\]](https://openreview.net/pdf?id=w79RtqIyoM) [\[code\]](https://github.com/timgaripov/compositional-sculpting)

* **Trans-Dimensional Generative Modeling via Jump Diffusion Models**, *NeurIPS*, 2023. [\[pdf\]](https://arxiv.org/pdf/2305.16261.pdf) [\[code\]](https://github.com/andrew-cr/jump-diffusion)

* **M2Hub: Unlocking the Potential of Machine Learning for Materials Discovery**, *NeurIPS*, 2023. [\[pdf\]](https://openreview.net/pdf?id=eJ5nu9qvWz) [\[code\]](https://github.com/yuanqidu/M2Hub)

* **LagrangeBench: A Lagrangian Fluid Mechanics Benchmarking Suite**, *NeurIPS*, 2023. [\[pdf\]](https://arxiv.org/pdf/2309.16342.pdf) [\[code\]](https://github.com/tumaer/lagrangebench)

* **ProteinInvBench: Benchmarking Protein Inverse Folding on Diverse Tasks, Models, and Metrics**, *NeurIPS*, 2023. [\[pdf\]](https://openreview.net/pdf?id=bqXduvuW5E) [\[code\]](https://github.com/A4Bio/ProteinInvBench)

* **OpenProteinSet: Training Data for Structural Biology at Scale**, *NeurIPS*, 2023. [\[pdf\]](https://openreview.net/pdf?id=gO0kS0eE0F) [\[code\]](https://github.com/aqlaboratory/openfold)

* **Sample-efficient Multi-objective Molecular Optimization with GFlowNets**, *NeurIPS*, 2023. [\[pdf\]](https://arxiv.org/pdf/2302.04040.pdf) [\[code\]](https://github.com/violet-sto/HN-GFN)

* **Graph Denoising Diffusion for Inverse Protein Folding**, *NeurIPS*, 2023. [\[pdf\]](https://arxiv.org/pdf/2306.16819.pdf) [\[code\]](https://github.com/ykiiiiii/GraDe_IF)

* **GAUCHE: A Library for Gaussian Processes in Chemistry**, *NeurIPS*, 2023. [\[pdf\]](https://openreview.net/pdf?id=vzrA6uqOis) [\[code\]](https://github.com/leojklarner/gauche)

* **What Can Large Language Models Do in Chemistry? A Comprehensive Benchmark on Eight Tasks**, *NeurIPS*, 2023. [\[pdf\]](https://arxiv.org/pdf/2305.18365.pdf) [\[code\]](https://github.com/ChemFoundationModels/ChemLLMBench)

* **TARTARUS: A Benchmarking Platform for Realistic And Practical Inverse Molecular Design**, *NeurIPS*, 2023. [\[pdf\]](https://arxiv.org/pdf/2209.12487.pdf) [\[code\]](https://github.com/aspuru-guzik-group/Tartarus)

* **DiffPack: A Torsional Diffusion Model for Autoregressive Protein Side-Chain Packing**, *NeurIPS*, 2023. [\[pdf\]](https://openreview.net/pdf?id=sXMQPKbLXf) [\[code\]](https://github.com/DeepGraphLearning/DiffPack)

* **ProteinShake Building Datasets and Benchmarks for Deep Learning on Protein Structures**, *NeurIPS*, 2023. [\[pdf\]](https://openreview.net/pdf?id=27vPcG4vKV) [\[code\]](https://proteinshake.ai/)

* **Uncovering Neural Scaling Laws in Molecular Representation Learning**, *NeurIPS*, 2023. [\[pdf\]](https://openreview.net/pdf?id=Ys8RmfF9w1) [\[code\]](https://github.com/Data-reindeer/MolScaling)

* **Benchmark of Machine Learning Force Fields for Semiconductor Simulations: Datasets, Metrics, and Comparative Analysis**, *NeurIPS*, 2023. [\[pdf\]]() [\[code\]]()

* **Lo-Hi: Practical ML Drug Discovery Benchmark**, *NeurIPS*, 2023. [\[pdf\]](https://openreview.net/pdf?id=hr9Bd1A9Un) [\[code\]](https://github.com/SAITPublic/MLFF-Framework)

* **Equivariant Flow Matching with Hybrid Probability Transport for 3D Molecule Generation**, *NeurIPS*, 2023. [\[pdf\]](https://arxiv.org/pdf/2312.07168.pdf) [\[code\]](https://github.com/AlgoMole/MolFM)

* **Learning Adaptive Tensorial Density Fields for Clean Cryo-ET Reconstruction**, *NeurIPS*, 2023. [\[pdf\]](https://openreview.net/pdf?id=r7g9nFsulw) [\[code\]](https://github.com/yuanhaowang1213/adaptivetensordf)

* **Improving Compositional Generalization using Iterated Learning and Simplicial Embeddings**, *NeurIPS*, 2023. [\[pdf\]](https://arxiv.org/pdf/2310.18777.pdf) 

* **Equivariant Spatio-Temporal Attentive Graph Networks to Simulate Physical Dynamics**, *NeurIPS*, 2023. [\[pdf\]](https://openreview.net/pdf?id=35nFSbEBks) [\[code\]](https://github.com/ManlioWu/ESTAG)

* **Clifford Group Equivariant Neural Networks**, *NeurIPS*, 2023. [\[pdf\]](https://arxiv.org/pdf/2305.11141.pdf) [\[code\]](https://github.com/DavidRuhe/clifford-group-equivariant-neural-networks)

* **A New Perspective on Building Efficient and Expressive 3D Equivariant Graph Neural Networks**, *NeurIPS*, 2023. [\[pdf\]](https://openreview.net/pdf?id=hWPNYWkYPN) [\[code\]](https://github.com/yuanqidu/LeftNet)

* **Metropolis Sampling for Constrained Diffusion Models**, *NeurIPS*, 2023. [\[pdf\]](https://arxiv.org/pdf/2307.05439.pdf) 

* **Multi-resolution Spectral Coherence for Graph Generation with Score-based Diffusion**, *NeurIPS*, 2023. [\[pdf\]](https://openreview.net/pdf?id=qUlpDjYnsp) 

* **Bootstrapped Training of Score-Conditioned Generator for Offline Design of Biological Sequences**, *NeurIPS*, 2023. [\[pdf\]]() [\[code\]](https://github.com/kaist-silab/bootgen)

* **Wasserstein Quantum Monte Carlo: A Novel Approach for Solving the Quantum Many-Body Schrödinger Equation**, *NeurIPS*, 2023. [\[pdf\]](https://arxiv.org/pdf/2307.07050.pdf) [\[code\]](https://github.com/necludov/wqmc)

* **Versatile Energy-Based Probabilistic Models for High Energy Physics**, *NeurIPS*, 2023. [\[pdf\]](https://arxiv.org/pdf/2302.00695.pdf) [\[code\]](https://github.com/taolicheng/EBM-HEP)

* **xTrimoGene: An Efficient and Scalable Representation Learner for Single-Cell RNA-Seq Data**, *NeurIPS*, 2023. [\[pdf\]](https://arxiv.org/pdf/2311.15156.pdf) 

* **Spatially Resolved Gene Expression Prediction from H&E Histology Images via Bi-modal Contrastive Learning**, *NeurIPS*, 2023. [\[pdf\]](https://arxiv.org/pdf/2306.01859.pdf) [\[code\]](https://github.com/bowang-lab/BLEEP)

* **Functional-Group-Based Diffusion for Pocket-Specific Molecule Generation and Elaboration**, *NeurIPS*, 2023. [\[pdf\]](https://arxiv.org/pdf/2306.13769.pdf) 

* **Learning to Group Auxiliary Datasets for Molecule**, *NeurIPS*, 2023. [\[pdf\]](https://arxiv.org/pdf/2307.04052.pdf) [\[code\]](https://github.com/Graph-and-Geometric-Learning/MolGroup)

* **QH9: A Quantum Hamiltonian Prediction Benchmark for QM9 Molecules**, *NeurIPS*, 2023. [\[pdf\]](https://arxiv.org/pdf/2306.09549.pdf) [\[code\]](https://github.com/divelab/AIRS/tree/main/OpenDFT/QHBench)

* **Learning Invariant Molecular Representation in Latent Discrete Space**, *NeurIPS*, 2023. [\[pdf\]](https://arxiv.org/pdf/2310.14170.pdf) [\[code\]](https://github.com/HICAI-ZJU/iMoLD)

* **DrugCLIP: Contrastive Protein-Molecule Representation Learning for Virtual Screening**, *NeurIPS*, 2023. [\[pdf\]](https://openreview.net/pdf?id=lAbCgNcxm7) [\[code\]](https://github.com/bowen-gao/DrugCLIP)

* **A Hierarchical Training Paradigm for Antibody Structure-sequence Co-design**, *NeurIPS*, 2023. [\[pdf\]](https://arxiv.org/pdf/2311.16126.pdf) [\[code\]](https://github.com/facebookresearch/esm)

* **Prefix-Tree Decoding for Predicting Mass Spectra from Molecules**, *NeurIPS*, 2023. [\[pdf\]](https://arxiv.org/pdf/2303.06470.pdf) [\[code\]](https://github.com/samgoldman97/ms-pred)

* **Rethinking Tokenizer and Decoder in Masked Graph Modeling for Molecules**, *NeurIPS*, 2023. [\[pdf\]](https://arxiv.org/pdf/2310.14753.pdf) [\[code\]](https://github.com/syr-cn/SimSGT)

* **GIMLET: A Unified Graph-Text Model for Instruction-Based Molecule Zero-Shot Learning**, *NeurIPS*, 2023. [\[pdf\]](https://arxiv.org/pdf/2306.13089.pdf) [\[code\]](https://github.com/zhao-ht/GIMLET)

* **QuinNet: Efficiently Incorporating Quintuple Interactions into Geometric Deep Learning Force Fields**, *NeurIPS*, 2023. [\[pdf\]](https://openreview.net/pdf?id=bPJmu1PbZD) [\[code\]](https://github.com/Zun-Wang/QuinNet)

* **The Geometry of Hidden Representations of Large Transformer Models**, *NeurIPS*, 2023. [\[pdf\]](https://arxiv.org/pdf/2302.00294.pdf) [\[code\]](https://github.com/diegodoimo/geometry_representations)

* **CELL-E 2: Translating Proteins to Pictures and Back with a Bidirectional Text-to-Image Transformer**, *NeurIPS*, 2023. [\[pdf\]](https://www.biorxiv.org/content/biorxiv/early/2023/10/07/2023.10.05.561066.full.pdf) [\[code\]](https://bohuanglab.github.io/CELL-E_2/)

* **Newton–Cotes Graph Neural Networks: On the Time Evolution of Dynamic Systems**, *NeurIPS*, 2023. [\[pdf\]](https://arxiv.org/pdf/2305.14642.pdf) [\[code\]](https://github.com/zjukg/NCGNN)

* **Equivariant Neural Simulators for Stochastic Spatiotemporal Dynamics**, *NeurIPS*, 2023. [\[pdf\]](https://arxiv.org/pdf/2305.14286.pdf) [\[code\]](https://github.com/kminartz/EPNS)

* **Predicting a Protein’s Stability under a Million Mutations**, *NeurIPS*, 2023. [\[pdf\]](https://arxiv.org/pdf/2310.12979.pdf) [\[code\]](https://github.com/jozhang97/MutateEverything)

* **Injecting Multimodal Information into Rigid Protein Docking via Bi-level Optimization**, *NeurIPS*, 2023. [\[pdf\]](https://openreview.net/pdf?id=ZuaVKlWdD2) 

* **Full-Atom Protein Pocket Design via Iterative Refinement**, *NeurIPS*, 2023. [\[pdf\]](https://arxiv.org/pdf/2310.02553.pdf) [\[code\]](https://github.com/zaixizhang/FAIR)

* **3D Molecule Generation by Denoising Voxel Grids**, *NeurIPS*, 2023. [\[pdf\]](https://openreview.net/pdf?id=Zyzluw0hC4) [\[code\]](https://github.com/cvignac/MiDi)

* **AVIDa-hIL6: A Large-Scale VHH Dataset Produced from an Immunized Alpaca for Predicting Antigen-Antibody Interactions**, *NeurIPS*, 2023. [\[pdf\]](https://openreview.net/pdf?id=BgY17iEnTb) [\[code\]](https://cognanous.com/datasets/avida-hil6)

* **Protein Design with Guided Discrete Diffusion**, *NeurIPS*, 2023. [\[pdf\]](https://arxiv.org/pdf/2305.20009.pdf) [\[code\]](https://github.com/ngruver/NOS)

* **Crystal Structure Prediction by Joint Equivariant Diffusion**, *NeurIPS*, 2023. [\[pdf\]](https://openreview.net/pdf?id=DNdN26m2Jk) [\[code\]](https://github.com/jiaor17/DiffCSP)

* **Understanding the Limitations of Deep Models for Molecular property prediction: Insights and Solutions**, *NeurIPS*, 2023. [\[pdf\]](https://openreview.net/pdf?id=NLFqlDeuzt) 

* **Tanimoto Random Features for Scalable Molecular Machine Learning**, *NeurIPS*, 2023. [\[pdf\]](https://arxiv.org/pdf/2306.14809.pdf) [\[code\]](https://github.com/AustinT/tanimoto-random-features-neurips23)

* **Estimating Koopman Operators with Sketching to Provably Learn Large Scale Dynamical Systems**, *NeurIPS*, 2023. [\[pdf\]](https://openreview.net/pdf?id=GItLpB1vhK) [\[code\]](https://github.com/Giodiro/NystromKoopman)

* **Geometric Algebra Transformer**, *NeurIPS*, 2023. [\[pdf\]](https://arxiv.org/pdf/2305.18415.pdf) [\[code\]](https://github.com/qualcomm-ai-research/geometric-algebra-transformer)

* **Is Distance Matrix Enough for Geometric Deep Learning?**, *NeurIPS*, 2023. [\[pdf\]](https://arxiv.org/pdf/2302.05743.pdf) 

* **Variational Monte Carlo on a Budget – Fine-tuning pre-trained Neural Wavefunctions**, *NeurIPS*, 2023. [\[pdf\]](https://openreview.net/pdf?id=FBNyccPfAu) [\[code\]](https://github.com/mdsunivie/deeperwin)

* **SE(3) Equivariant Augmented Coupling Flows**, *NeurIPS*, 2023. [\[pdf\]](https://arxiv.org/pdf/2308.10364.pdf) [\[code\]](https://github.com/lollcat/se3-augmented-coupling-flows)

* **Learning Time-Invariant Representations for Individual Neurons from Population Dynamics**, *NeurIPS*, 2023. [\[pdf\]](https://arxiv.org/pdf/2311.02258.pdf) [\[code\]](https://github.com/lumimim/NeuPRINT/)

* **ProteinGym: Large-Scale Benchmarks for Protein Fitness Prediction and Design**, *NeurIPS*, 2023. [\[pdf\]](https://openreview.net/pdf?id=URoZHqAohf) [\[code\]](https://github.com/OATML-Markslab/ProteinGym)

* **HyenaDNA: Long-Range Genomic Sequence Modeling at Single Nucleotide Resolution**, *NeurIPS*, 2023. [\[pdf\]](https://openreview.net/pdf?id=ubzNoJjOKj) [\[code\]](https://github.com/HazyResearch/hyena-dna)

* **ResoNet: Noise-Trained Physics-Informed MRI Off-Resonance Correction**, *NeurIPS*, 2023. [\[pdf\]](https://openreview.net/pdf?id=Ia4dmqst0Z) [\[code\]](https://github.com/mikgroup/ResoNet)

* **DSR: Dynamical Surface Representation as Implicit Neural Networks for Protein**, *NeurIPS*, 2023. [\[pdf\]](https://openreview.net/pdf?id=IOSaJ7ukgf) [\[code\]](https://sundw-818.github.io/DSR/)

* **Transfer Learning for Atomistic Simulations Using GNNs and Kernel Mean Embeddings**, *NeurIPS*, 2023. [\[pdf\]](https://arxiv.org/pdf/2306.01589.pdf) [\[code\]](https://github.com/IsakFalk/atomistic_transfer_mekrr)

* **FABind: Fast and Accurate Protein-Ligand Binding**, *NeurIPS*, 2023. [\[pdf\]](https://arxiv.org/pdf/2310.06763.pdf) [\[code\]](https://github.com/QizhiPei/FABind)

* **ProBio: A Protocol-guided Multimodal Dataset for Molecular Biology Lab**, *NeurIPS*, 2023. [\[pdf\]](https://arxiv.org/pdf/2311.00556.pdf) [\[code\]](https://probio-dataset.github.io/)

* **CoDrug: Conformal Drug Property Prediction with Density Estimation under Covariate Shift**, *NeurIPS*, 2023. [\[pdf\]](https://arxiv.org/pdf/2310.12033.pdf) 

* **Towards Symmetry-Aware Generation of Periodic Materials**, *NeurIPS*, 2023. [\[pdf\]](https://arxiv.org/pdf/2307.02707.pdf) [\[code\]](https://github.com/divelab/AIRS)

* **Timewarp: Transferable Acceleration of Molecular Dynamics by Learning Time-Coarsened Dynamics**, *NeurIPS*, 2023. [\[pdf\]](https://arxiv.org/pdf/2302.01170.pdf) [\[code\]](https://github.com/microsoft/timewarp)

* **Equivariant Neural Operator Learning with Graphon Convolution**, *NeurIPS*, 2023. [\[pdf\]](https://arxiv.org/pdf/2311.10908.pdf) [\[code\]](https://github.com/ccr-cheng/InfGCN-pytorch)

* **Stochastic Optimal Control for Collective Variable Free Sampling of Molecular Transition Paths**, *NeurIPS*, 2023. [\[pdf\]](https://openreview.net/pdf?id=LUVqEs90mq) [\[code\]](https://github.com/LarsHoldijk/SOCTransitionPaths)

* **Symmetry-Informed Geometric Representation for Molecules, Proteins, and Crystalline Materials**, *NeurIPS*, 2023. [\[pdf\]](https://openreview.net/pdf?id=ygXSNrIU1p) [\[code\]](https://github.com/chao1224/Geom3D)
