# Drug discovery papers

This repository contains a list of papers on molecular and protein, we categorize them based on their tasks and model types.

We will try to make this list updated. If you found any error or any missed paper, please don't hesitate to open an issue or pull request.

* **BASC: Applying Bayesian Optimization to the Search for Global Minima on Potential Energy Surfaces**, *ICML*, 2016. [\[pdf\]](http://proceedings.mlr.press/v48/carr16.pdf)   

* **Protein Contact Prediction from Amino Acid Co-evolution using Convolutional Networks for Graph-valued Images**, *NeurIPS*, 2016. [\[pdf\]](https://proceedings.neurips.cc/paper_files/paper/2016/file/2cad8fa47bbef282badbb8de5374b894-Paper.pdf)  

* **Neural Message Passing for Quantum Chemistry**, *ICML*, 2017. [\[pdf\]](https://arxiv.org/pdf/1704.01212.pdf)  

* **Parallel and Distributed Thompson Sampling for Large-scale Accelerated Exploration of Chemical Space**, *ICML*, 2017. [\[pdf\]](https://arxiv.org/pdf/1706.01825.pdf)  

* **Variational Inference for Sparse and Undirected Models**, *ICML*, 2017. [\[pdf\]](https://arxiv.org/pdf/1602.03807.pdf)  

* **Sequence Tutor: Conservative Fine-Tuning of Sequence Generation Models with KL-control**, *ICML*, 2017. [\[pdf\]](https://arxiv.org/pdf/1611.02796.pdf) [\[code\]](https://github.com/magenta/magenta/tree/main/magenta/models/rl_tuner)   

* **Grammar Variational Autoencoder**, *ICML*, 2017. [\[pdf\]](https://arxiv.org/pdf/1703.01925.pdf)  

* **Deriving Neural Architectures from Sequence and Graph Kernels**, *ICML*, 2017. [\[pdf\]](https://arxiv.org/pdf/1705.09037.pdf) [\[code\]](https://github.com/taolei87/icml17_knn)  

* **SchNet: A Continuous-filter Convolutional Neural Network for Modeling Quantum Interactions**, *NeurIPS*, 2017. [\[pdf\]](https://arxiv.org/pdf/1706.08566.pdf)    

* **Solid Harmonic Wavelet Scattering: Predicting Quantum Molecular Energy from Invariant Descriptors of 3D Electronic Densities**, *NeurIPS*, 2017. [\[pdf\]](https://proceedings.neurips.cc/paper_files/paper/2017/file/72b386224056bf940cd5b01341f65e9d-Paper.pdf)    

* **Spherical Convolutions and Their Application in Molecular Modelling**, *NeurIPS*, 2017. [\[pdf\]](https://papers.nips.cc/paper_files/paper/2017/file/1113d7a76ffceca1bb350bfe145467c6-Paper.pdf) [\[code\]](https://github.com/deepfold/NIPS2017)  

* **Predicting Organic Reaction Outcomes with Weisfeiler-Lehman Network**, *NeurIPS*, 2017. [\[pdf\]](https://arxiv.org/pdf/1709.04555.pdf) [\[code\]](https://github.com/wengong-jin/nips17-rexgen)       

* **Protein Interface Prediction using Graph Convolutional Networks**, *NeurIPS*, 2017. [\[pdf\]](https://proceedings.neurips.cc/paper_files/paper/2017/file/f507783927f2ec2737ba40afbd17efb5-Paper.pdf) [\[code\]](https://github.com/fouticus/pipgcn)        
* **Learning a Generative Model for Validity in Complex Discrete Structures**, *ICLR*, 2018. [\[pdf\]](https://arxiv.org/pdf/1712.01664.pdf) [\[code\]](https://github.com/DavidJanz/molecule_grammar_rnn)         

* **Syntax-Directed Variational Autoencoder for Structured Data**, *ICLR*, 2018. [\[pdf\]](https://arxiv.org/pdf/1802.08786.pdf) [\[code\]](https://github.com/Hanjun-Dai/sdvae)    

* **Junction Tree Variational Autoencoder for Molecular Graph Generation**, *ICML*, 2018. [\[pdf\]](https://arxiv.org/pdf/1802.04364.pdf) [\[code\]](https://github.com/wengong-jin/icml18-jtnn)        

* **GraphRNN: Generating Realistic Graphs with Deep Auto-regressive Models**, *ICML*, 2018. [\[pdf\]](https://arxiv.org/pdf/1802.08773.pdf) [\[code\]](https://github.com/snap-stanford/GraphRNN)        

* **Generative Modeling for Protein Structures**, *NeurIPS*, 2018. [\[pdf\]](https://proceedings.neurips.cc/paper_files/paper/2018/file/afa299a4d1d8c52e75dd8a24c3ce534f-Paper.pdf)    

* **Constrained Graph Variational Autoencoders for Molecule Design**, *NeurIPS*, 2018. [\[pdf\]](https://arxiv.org/pdf/1805.09076.pdf) [\[code\]](https://github.com/Microsoft/constrained-graph-variational-autoencoder)   

* **Neural Edit Operations for Biological Sequences**, *NeurIPS*, 2018. [\[pdf\]](https://papers.nips.cc/paper_files/paper/2018/file/d0921d442ee91b896ad95059d13df618-Paper.pdf)    

* **Graph Convolutional Policy Network for Goal-Directed Molecular Graph Generation**, *NeurIPS*, 2018. [\[pdf\]](https://arxiv.org/pdf/1806.02473.pdf) [\[code\]](https://github.com/bowenliu16/rl_graph_generation)   

* **Deep Generative Markov State Models**, *NeurIPS*, 2018. [\[pdf\]](https://arxiv.org/pdf/1805.07601.pdf) [\[code\]](https://github.com/markovmodel/deep_gen_msm)   

* **Constrained Generation of Semantically Valid Graphs via Regularizing Variational Autoencoders**, *NeurIPS*, 2018. [\[pdf\]](https://arxiv.org/pdf/1809.02630.pdf)    

* **End-to-end Symmetry Preserving Inter-atomic Potential Energy Model for Finite and Extended Systems**, *NeurIPS*, 2018. [\[pdf\]](https://arxiv.org/pdf/1805.09003.pdf)       

* **Learning Protein Sequence Embeddings using Information from Structure**, *ICLR*, 2019. [\[pdf\]](https://arxiv.org/pdf/1902.08661.pdf) [\[code\]](https://github.com/tbepler/protein-sequence-embedding-iclr2019)   

* **Learning Multimodal Graph-to-Graph Translation for Molecular Optimization**, *ICLR*, 2019. [\[pdf\]](https://arxiv.org/pdf/1812.01070.pdf)    

* **A Generative Model For Electron Paths**, *ICLR*, 2019. [\[pdf\]](https://arxiv.org/pdf/1805.10970.pdf)   

* **Human-level Protein Localization with Convolutional Neural Networks**, *ICLR*, 2019. [\[pdf\]](https://openreview.net/pdf?id=ryl5khRcKm) [\[code\]](https://github.com/ml-jku/gapnet-pl)  

* **Learning Protein Structure with a Differentiable Simulator**, *ICLR*, 2019. [\[pdf\]](https://openreview.net/pdf?id=Byg3y3C9Km)   

* **Molecular Hypergraph Grammar with Its Application to Molecular Optimization**, *ICML*, 2019. [\[pdf\]](http://proceedings.mlr.press/v97/kajino19a/kajino19a.pdf)   

* **Functional Transparency for Structured Data: a Game-Theoretic Approach**, *ICML*, 2019. [\[pdf\]](https://arxiv.org/pdf/1902.09737.pdf)   

* **DeepNose: Using Artificial Neural Networks to Represent the Space of Odorants**, *ICML*, 2019. [\[pdf\]](http://proceedings.mlr.press/v97/tran19b/tran19b.pdf)   

* **Conditioning by Adaptive Sampling for Robust Design**, *ICML*, 2019. [\[pdf\]]() [\[code\]]()  

* **Efficient Graph Generation with Graph Recurrent Attention Network**, *ICML*, 2019. [\[pdf\]](https://arxiv.org/pdf/1910.00760.pdf) [\[code\]](https://github.com/lrjconan/GRAN)  

* **Learning to Optimize in Swarms**, *NeurIPS*, 2019. [\[pdf\]](https://openreview.net/pdf?id=rke8aHHlIH) [\[code\]](https://github.com/Shen-Lab/LOIS)  

* **Generative Models for Graph-based Protein Design**, *NeurIPS*, 2019. [\[pdf\]](https://papers.nips.cc/paper_files/paper/2019/file/f3a4ff4839c56a5f460c88cce3666a2b-Paper.pdf) [\[code\]](https://github.com/jingraham/neurips19-graph-protein-design)  

* **Discrete Object Generation with Reversible Inductive Construction**, *NeurIPS*, 2019. [\[pdf\]](https://arxiv.org/pdf/1907.08268.pdf) [\[code\]](https://github.com/PrincetonLIPS/reversible-inductive-construction)  

* **Graph Normalizing Flow**, *NeurIPS*, 2019. [\[pdf\]](https://arxiv.org/pdf/1905.13177.pdf)   

* **Cormorant: Covariant Molecular Neural Networks**, *NeurIPS*, 2019. [\[pdf\]](https://arxiv.org/pdf/1906.04015.pdf) [\[code\]](https://github.com/risilab/cormorant)  

* **Integrating Markov Processes with Structural Causal Modeling Enables Counterfactual Inference in Complex Systems**, *NeurIPS*, 2019. [\[pdf\]](https://arxiv.org/pdf/1911.02175.pdf) 

* **N-Gram Graph: Simple Unsupervised Representation for Graphs, with Applications to Molecules**, *NeurIPS*, 2019. [\[pdf\]](https://arxiv.org/pdf/1806.09206.pdf) [\[code\]](https://github.com/chao1224/n_gram_graph)

* **Evaluating Protein Transfer Learning with TAPE**, *NeurIPS*, 2019. [\[pdf\]](https://arxiv.org/pdf/1906.08230.pdf) [\[code\]](https://github.com/songlab-cal/tape)

* **A Model to Search for Synthesizable Molecules**, *NeurIPS*, 2019. [\[pdf\]](https://arxiv.org/pdf/1906.05221.pdf) [\[code\]](https://github.com/john-bradshaw/molecule-chef)

* **Depth-First Proof-Number Search with Heuristic Edge Cost and Application to Chemical Synthesis Planning**, *NeurIPS*, 2019. [\[pdf\]](https://proceedings.neurips.cc/paper_files/paper/2019/file/4fc28b7093b135c21c7183ac07e928a6-Paper.pdf) 

* **End-to-End Learning on 3D Protein Structure for Interface Prediction**, *NeurIPS*, 2019. [\[pdf\]](https://arxiv.org/pdf/1807.01297.pdf) [\[code\]](https://github.com/drorlab/DIPS)  

* **Retrosynthesis Prediction with Conditional Graph Logic Network**, *NeurIPS*, 2019. [\[pdf\]](https://arxiv.org/pdf/2001.01408.pdf) [\[code\]](https://github.com/Hanjun-Dai/GLN)  

* **Symmetry-adapted Generation of 3D Point Sets for the Targeted Discovery of Molecules**, *NeurIPS*, 2019. [\[pdf\]](https://arxiv.org/pdf/1906.00957.pdf) [\[code\]](https://github.com/atomistic-machine-learning/G-SchNet)  

* **Hyperbolic Graph Neural Networks**, *NeurIPS*, 2019. [\[pdf\]](https://arxiv.org/pdf/1910.12892.pdf) [\[code\]](https://github.com/facebookresearch/hgnn)  

* **Deep imitation learning for molecular inverse problems**, *NeurIPS*, 2019. [\[pdf\]](https://proceedings.neurips.cc/paper/2019/file/b0bef4c9a6e50d43880191492d4fc827-Paper.pdf)  

* **From Variational to Deterministic Autoencoders**, *ICLR*, 2020. [\[pdf\]](https://arxiv.org/pdf/1903.12436.pdf) [\[code\]](https://github.com/ParthaEth/Regularized_autoencoders-RAE-)  

* **Augmenting Genetic Algorithms with Deep Neural Networks for Exploring the Chemical Space**, *ICLR*, 2020. [\[pdf\]](https://arxiv.org/pdf/1909.11655.pdf) [\[code\]](https://github.com/aspuru-guzik-group/GA)  

* **Oblique Decision Trees from Derivatives of ReLU Networks**, *ICLR*, 2020. [\[pdf\]](https://arxiv.org/pdf/1909.13488.pdf) [\[code\]](https://github.com/guanghelee/iclr20-lcn)  

* **GraphAF: a Flow-based Autoregressive Model for Molecular Graph Generation**, *ICLR*, 2020. [\[pdf\]](https://arxiv.org/pdf/2001.09382.pdf) [\[code\]](https://github.com/DeepGraphLearning/GraphAF)  

* **Model-based Reinforcement Learning for Biological Sequence Design**, *ICLR*, 2020. [\[pdf\]](https://openreview.net/pdf?id=HklxbgBKvr)   

* **InfoGraph: Unsupervised and Semi-supervised Graph-Level Representation Learning via Mutual Information Maximization**, *ICLR*, 2020. [\[pdf\]](https://arxiv.org/pdf/1908.01000.pdf)   

* **Strategies for Pre-training Graph Neural Networks**, *ICLR*, 2020. [\[pdf\]](https://arxiv.org/pdf/1905.12265.pdf)   

* **Directional Message Passing for Molecular Graphs**, *ICLR*, 2020. [\[pdf\]](https://arxiv.org/pdf/2003.03123.pdf)   

* **Energy-based Models for Atomic-resolution Protein Conformations**, *ICLR*, 2020. [\[pdf\]]() [\[code\]](https://github.com/facebookresearch/protein-ebm)  

* **Reconstructing Continuous Distributions of 3D Protein Structure from Cryo-EM Images**, *ICLR*, 2020. [\[pdf\]](https://arxiv.org/pdf/1909.05215.pdf)   

* **RNA Secondary Structure Prediction By Learning Unrolled Algorithms**, *ICLR*, 2020. [\[pdf\]](https://arxiv.org/pdf/2002.05810.pdf) [\[code\]](https://github.com/ml4bio/e2efold)  

* **Retro-star: Learning Retrosynthetic Planning with Neural Guided A-star Search**, *ICML*, 2020. [\[pdf\]](https://arxiv.org/pdf/2006.15820.pdf) [\[code\]](https://github.com/binghong-ml/retro_star)  

* **Scalable Deep Generative Modeling for Sparse Graphs**, *ICML*, 2020. [\[pdf\]](https://arxiv.org/pdf/2006.15502.pdf) [\[code\]](https://github.com/google-research/google-research/tree/master/bigg)  

* **Generalizing Convolutional Neural Networks for Equivariance to Lie Groups on Arbitrary Continuous Data**, *ICML*, 2020. [\[pdf\]](https://arxiv.org/pdf/2002.12880.pdf) [\[code\]](https://github.com/mfinzi/LieConv)  

* **Population-Based Black-Box Optimization for Biological Sequence Design**, *ICML*, 2020. [\[pdf\]](https://arxiv.org/pdf/2006.03227.pdf)   

* **Learning to Navigate The Synthetically Accessible Chemical Space Using Reinforcement Learning**, *ICML*, 2020. [\[pdf\]](http://proceedings.mlr.press/v119/gottipati20a/gottipati20a.pdf) [\[code\]](https://github.com/99andBeyond/Apollo1060)  

* **Hierarchical Generation of Molecular Graphs using Structural Motifs**, *ICML*, 2020. [\[pdf\]](https://arxiv.org/pdf/2002.03230.pdf)   

* **Multi-Objective Molecule Generation using Interpretable Substructures**, *ICML*, 2020. [\[pdf\]](https://arxiv.org/pdf/2002.03244.pdf) [\[code\]](https://github.com/wengong-jin/multiobj-rationale)  

* **Online Dense Subgraph Discovery via Blurred-Graph Feedback**, *ICML*, 2020. [\[pdf\]](https://arxiv.org/pdf/2006.13642.pdf)   

* **A Chance-Constrained Generative Framework for Sequence Optimization**, *ICML*, 2020. [\[pdf\]](http://proceedings.mlr.press/v119/liu20i/liu20i.pdf)   

* **Projective Preferential Bayesian Optimization**, *ICMLR*, 2020. [\[pdf\]](https://arxiv.org/pdf/2002.03113.pdf) [\[code\]](https://github.com/AaltoPML/PPBO)  

* **A Graph to Graphs Framework for Retrosynthesis Prediction**, *ICML*, 2020. [\[pdf\]](https://arxiv.org/pdf/2003.12725.pdf)   

* **A Generative Model for Molecular Distance Geometry**, *ICML*, 2020. [\[pdf\]](https://arxiv.org/pdf/1909.11459.pdf) [\[code\]](https://github.com/gncs/graphdg)  

* **Reinforcement Learning for Molecular Design Guided by Quantum Mechanics**, *ICML*, 2020. [\[pdf\]](https://arxiv.org/pdf/2002.07717.pdf) [\[code\]](https://github.com/gncs/molgym)  

* **Improving Molecular Design by Stochastic Iterative Target Augmentation**, *ICML*, 2020. [\[pdf\]](https://arxiv.org/pdf/2002.04720.pdf) [\[code\]](https://github.com/yangkevin2/icml2020-stochastic-iterative-target-augmentation)  

* **A Graph Similarity for Deep Learning**, *NeurIPS*, 2020. [\[pdf\]](https://papers.nips.cc/paper/2020/file/0004d0b59e19461ff126e3a08a814c33-Paper.pdf) [\[code\]](https://github.com/se-ok/WLsimilarity)  

* **Guiding Deep Molecular Optimization with Genetic Exploration**, *NeurIPS*, 2020. [\[pdf\]](https://arxiv.org/pdf/2007.04897.pdf) [\[code\]](https://github.com/sungsoo-ahn/genetic-expert-guided-learning)  

* **Self-Supervised Graph Transformer on Large-Scale Molecular Data**, *NeurIPS*, 2020. [\[pdf\]](https://arxiv.org/pdf/2007.02835.pdf)   

* **Building Powerful and Equivariant Graph Neural Networks with Structural Message-passing**, *NeurIPS*, 2020. [\[pdf\]]() [\[code\]]()  

* **Efficient Generation of Structured Objects with Constrained Adversarial Networks**, *NeurIPS*, 2020. [\[pdf\]](https://arxiv.org/pdf/2007.13197.pdf) [\[code\]](https://github.com/unitn-sml/CAN)  

* **BOSS: Bayesian Optimization over String Spaces**, *NeurIPS*, 2020. [\[pdf\]](https://arxiv.org/pdf/2010.00979.pdf) [\[code\]](https://github.com/henrymoss/BOSS) 

* **Learning Invariances in Neural Networks**, *NeurIPS*, 2020. [\[pdf\]](https://arxiv.org/pdf/2010.11882.pdf) [\[code\]](https://github.com/g-benton/learning-invariances) 

* **Inverse Learning of Symmetries**, *NeurIPS*, 2020. [\[pdf\]](https://arxiv.org/pdf/2002.02782.pdf) [\[code\]](https://github.com/bmda-unibas/InverseLearningOfSymmetries) 

* **Provable Overlapping Community Detection in Weighted Graphs**, *NeurIPS*, 2020. [\[pdf\]](https://arxiv.org/pdf/2004.07150.pdf)  

* **BayReL: Bayesian Relational Learning for Multi-omics Data Integration**, *NeurIPS*, 2020. [\[pdf\]](https://arxiv.org/pdf/2010.05895.pdf)  

* **SE(3)-Transformers: 3D Roto-Translation Equivariant Attention Networks**, *NeurIPS*, 2020. [\[pdf\]](https://arxiv.org/pdf/2006.10503.pdf) [\[code\]](https://github.com/FabianFuchsML/se3-transformer-public)  

* **On the Equivalence of Molecular Graph Convolution and Molecular Wave Function with Poor Basis Set**, *NeurIPS*, 2020. [\[pdf\]](https://arxiv.org/pdf/2011.07929.pdf) [\[code\]](https://github.com/masashitsubaki/QuantumDeepField_molecule) 

* **TorsionNet: A Reinforcement Learning Approach to Sequential Conformer Search**, *NeurIPS*, 2020. [\[pdf\]](https://proceedings.neurips.cc/paper/2020/file/e904831f48e729f9ad8355a894334700-Paper.pdf) [\[code\]](https://github.com/tarungog/torsionnet_paper_version) 

* **Weisfeiler and Leman Go Sparse: Towards Scalable Higher-order Graph Embeddings**, *NeurIPS*, 2020. [\[pdf\]](https://arxiv.org/pdf/1904.01543.pdf) [\[code\]](https://github.com/chrsmrrs/sparsewl)  

* **Goal-directed Generation of Discrete Structures with Conditional Generative Models**, *NeurIPS*, 2020. [\[pdf\]](https://arxiv.org/pdf/2010.02311.pdf)  

* **CogMol: Target-Specific and Selective Drug Design for COVID-19 Using Deep Generative Models**, *NeurIPS*, 2020. [\[pdf\]](https://arxiv.org/pdf/2004.01215.pdf)  

* **Evaluating Attribution for Graph Neural Networks**, *NeurIPS*, 2020. [\[pdf\]](https://papers.nips.cc/paper_files/paper/2020/file/417fbbf2e9d5a28a855a11894b2e795a-Paper.pdf) [\[code\]](https://github.com/google-research/graph-attribution) 

* **Stochastic Normalizing Flows**, *NeurIPS*, 2020. [\[pdf\]](https://arxiv.org/pdf/2002.06707.pdf) [\[code\]](https://github.com/noegroup/stochastic_normalizing_flows) 

* **Barking up the Right Tree: An Approach to Search over Molecule Synthesis DAGs**, *NeurIPS*, 2020. [\[pdf\]](https://arxiv.org/pdf/2012.11522.pdf) [\[code\]](https://github.com/john-bradshaw/synthesis-dags) 

* **Learning Mutational Semantics**, *NeurIPS*, 2020. [\[pdf\]](https://proceedings.neurips.cc/paper_files/paper/2020/file/6754e06e46dfa419d5afe3c9781cecad-Paper.pdf) [\[code\]](https://github.com/brianhie/mutational-semantics-neurips2020) 

* **RetroXpert: Decompose Retrosynthesis Prediction Like A Chemist**, *NeurIPS*, 2020. [\[pdf\]](https://arxiv.org/pdf/2011.02893.pdf) [\[code\]](https://github.com/uta-smile/RetroXpert) 

* **Learning from Protein Structure with Geometric Vector Perceptrons**, *ICLR*, 2021. [\[pdf\]](https://arxiv.org/pdf/2009.01411.pdf) [\[code\]](https://github.com/drorlab/gvp) 

* **MARS: Markov Molecular Sampling for Multi-objective Drug Discovery**, *ICLR*, 2021. [\[pdf\]](https://arxiv.org/pdf/2103.10432.pdf) [\[code\]](https://github.com/bytedance/markov-molecular-sampling) 

* **Practical Massively Parallel Monte-Carlo Tree Search Applied to Molecular Design**, *ICLR*, 2021. [\[pdf\]](https://arxiv.org/pdf/2006.10504.pdf) [\[code\]](https://github.com/yoshizoe/mp-chemts)  

* **Learning Neural Generative Dynamics for Molecular Conformation Generation**, *ICLR*, 2021. [\[pdf\]](https://arxiv.org/pdf/2102.10240.pdf) [\[code\]](https://github.com/DeepGraphLearning/CGCF-ConfGen)  

* **Categorical Normalizing Flows via Continuous Transformations**, *ICLR*, 2021. [\[pdf\]](https://arxiv.org/pdf/2006.09790.pdf) [\[code\]](https://github.com/phlippe/CategoricalNF)  

* **Offline Model-Based Optimization via Normalized Maximum Likelihood Estimation**, *ICLR*, 2021. [\[pdf\]](https://arxiv.org/pdf/2102.07970.pdf)   

* **A Trainable Optimal Transport Embedding for Feature Aggregation and its Relationship to Attention**, *ICLR*, 2021. [\[pdf\]](https://arxiv.org/pdf/2006.12065.pdf) [\[code\]](https://github.com/claying/OTK)  

* **Intrinsic-Extrinsic Convolution and Pooling for Learning on 3D Protein Structures**, *ICLR*, 2021. [\[pdf\]](https://arxiv.org/pdf/2007.06252.pdf) [\[code\]](https://github.com/phermosilla/IEConv_proteins)  

* **HamNet: Conformation-Guided Molecular Representation with Hamiltonian Neural Networks**, *ICLR*, 2021. [\[pdf\]](https://arxiv.org/pdf/2105.03688.pdf)   

* **Lossless Compression of Structured Convolutional Models via Lifting**, *ICLR*, 2021. [\[pdf\]](https://arxiv.org/pdf/2007.06567.pdf)   

* **Property Controllable Variational Autoencoder via Invertible Mutual Dependence**, *ICLR*, 2021. [\[pdf\]](https://openreview.net/pdf?id=tYxG_OMs9WE) [\[code\]](https://github.com/xguo7/PCVAE)  

* **Graph Information Bottleneck for Subgraph Recognition**, *ICLR*, 2021. [\[pdf\]](https://openreview.net/pdf?id=bM4Iqfg8M2k) [\[code\]]()  

* **Transformer Protein Language Models Are Unsupervised Structure Learners**, *ICLR*, 2021. [\[pdf\]](https://openreview.net/pdf?id=fylclEqgvgd) [\[code\]](https://github.com/facebookresearch/esm)  

* **Symmetry-Aware Actor-Critic for 3D Molecular Design**, *ICLR*, 2021. [\[pdf\]](https://openreview.net/pdf?id=jEYKjPE1xYN)   

* **BERTology Meets Biology: Interpreting Attention in Protein Language Models**, *ICLR*, 2021. [\[pdf\]](https://arxiv.org/pdf/2006.15222.pdf) [\[code\]](https://github.com/salesforce/provis)  

* **Neural Representation and Generation for RNA Secondary Structures**, *ICLR*, 2021. [\[pdf\]](https://arxiv.org/pdf/2102.00925.pdf)   

* **Molecule Optimization by Explainable Evolution**, *ICLR*, 2021. [\[pdf\]](https://openreview.net/pdf?id=jHefDGsorp5) [\[code\]](https://github.com/binghong-ml/MolEvol)  

* **Learning to Make Decisions via Submodular Regularization**, *ICLR*, 2021. [\[pdf\]](https://openreview.net/pdf?id=ac288vnG_7U)   

* **Accurate Learning of Graph Representations with Graph Multiset Pooling**, *ICLR*, 2021. [\[pdf\]](https://arxiv.org/pdf/2102.11533.pdf) [\[code\]](https://github.com/JinheonBaek/GMT)  

* **A Structured Observation Distribution for Generative Biological Sequence Prediction and Forecasting**, *ICML*, 2021. [\[pdf\]](http://proceedings.mlr.press/v139/weinstein21a/weinstein21a.pdf)   

* **An End-to-End Framework for Molecular Conformation Generation via Bilevel Programming**, *ICML*, 2021. [\[pdf\]](https://arxiv.org/pdf/2105.07246.pdf) [\[code\]](https://github.com/MinkaiXu/ConfVAE-ICML21)  

* **Self-supervised Graph-level Representation Learning with Local and Global Structure**, *ICML*, 2021. [\[pdf\]](https://arxiv.org/pdf/2106.04113.pdf) [\[code\]](https://github.com/DeepGraphLearning/GraphLoG)  

* **Fold2Seq: A Joint Sequence(1D)-Fold(3D) Embedding-based Generative Model for Protein Design**, *ICML*, 2021. [\[pdf\]](https://arxiv.org/pdf/2106.13058.pdf) [\[code\]](https://github.com/IBM/fold2seq)  

* **XOR-CD: Linearly Convergent Constrained Structure Generatio**, *ICML*, 2021. [\[pdf\]](http://proceedings.mlr.press/v139/ding21a/ding21a.pdf)   

* **Bayesian Quadrature on Riemannian Data Manifolds**, *ICML*, 2021. [\[pdf\]](https://arxiv.org/pdf/2102.06645.pdf) [\[code\]](https://github.com/froec/BQonRDM)  

* **Improving Molecular Graph Neural Network Explainability with Orthonormalization and Induced Sparsity**, *ICML*, 2021. [\[pdf\]](https://arxiv.org/pdf/2105.04854.pdf) [\[code\]](https://github.com/bayer-science-for-a-better-life/graph-attribution) 

* **LieTransformer: Equivariant Self-Attention for Lie Groups**, *ICML*, 2021. [\[pdf\]](https://arxiv.org/pdf/2012.10885.pdf) [\[code\]](https://github.com/oxcsml/lie-transformer) 

* **Self-Improved Retrosynthetic Planning**, *ICML*, 2021. [\[pdf\]](https://arxiv.org/pdf/2106.04880.pdf)  

* **WILDS: A Benchmark of in-the-Wild Distribution Shifts**, *ICML*, 2021. [\[pdf\]](https://arxiv.org/pdf/2012.07421.pdf)  

* **GraphDF: A Discrete Flow Model for Molecular Graph Generation**, *ICML*, 2021. [\[pdf\]](https://arxiv.org/pdf/2102.01189.pdf)  

* **MSA Transformer**, *ICML*, 2021. [\[pdf\]](http://proceedings.mlr.press/v139/rao21a/rao21a.pdf) [\[code\]](https://github.com/facebookresearch/esm) 

* **Non-Autoregressive Electron Redistribution Modeling for Reaction Prediction**, *ICML*, 2021. [\[pdf\]](https://arxiv.org/pdf/2106.07801.pdf)  

* **E(n) Equivariant Graph Neural Networks**, *ICML*, 2021. [\[pdf\]](https://arxiv.org/pdf/2102.09844.pdf) [\[code\]](https://github.com/vgsatorras/egnn) 

* **Equivariant Message Passing for the Prediction of Tensorial Properties and Molecular Spectra**, *ICML*, 2021. [\[pdf\]](http://proceedings.mlr.press/v139/schutt21a/schutt21a.pdf)  

* **Learning Gradient Fields for Molecular Conformation Generation**, *ICML*, 2021. [\[pdf\]](https://arxiv.org/pdf/2105.03902.pdf) [\[code\]](https://github.com/DeepGraphLearning/ConfGF) 

* **Efficient Generative Modelling of Protein Structure Fragments using a Deep Markov Model**, *ICML*, 2021. [\[pdf\]](http://proceedings.mlr.press/v139/thygesen21a/thygesen21a.pdf)  

* **GEOMOL: Torsional Geometric Generation of Molecular 3D Conformer Ensembles**, *NeurIPS*, 2021. [\[pdf\]](https://arxiv.org/pdf/2106.07802.pdf) [\[code\]](https://github.com/PattanaikL/GeoMol) 

* **Towards Understanding Retrosynthesis by Energy-based Models**, *NeurIPS*, 2021. [\[pdf\]](https://arxiv.org/pdf/2007.13437.pdf)  

* **Learning Graph Models for Retrosynthesis Prediction**, *NeurIPS*, 2021. [\[pdf\]](https://arxiv.org/pdf/2006.07038.pdf) [\[code\]](https://github.com/vsomnath/graphretro) 

* **Functionally Regionalized Knowledge Transfer for Low-resource Drug Discovery**, *NeurIPS*, 2021. [\[pdf\]](https://papers.nips.cc/paper/2021/file/459a4ddcb586f24efd9395aa7662bc7c-Paper.pdf)  

* **Combining Latent Space and Structured Kernels for Bayesian Optimization over Combinatorial Spaces**, *NeurIPS*, 2021. [\[pdf\]](https://arxiv.org/pdf/2111.01186.pdf) [\[code\]](https://github.com/aryandeshwal/LADDER) 

* **Improving Black-box Optimization in VAE Latent Space using Decoder Uncertainty**, *NeurIPS*, 2021. [\[pdf\]](https://arxiv.org/pdf/2107.00096.pdf)  

* **Hit and Lead Discovery with Explorative RL and Fragment-based Molecule Generation**, *NeurIPS*, 2021. [\[pdf\]](https://arxiv.org/pdf/2110.01219.pdf) [\[code\]](https://github.com/AITRICS/FREED) 

* **Edge Representation Learning with Hypergraphs**, *NeurIPS*, 2021. [\[pdf\]](https://arxiv.org/pdf/2106.15845.pdf) [\[code\]](https://github.com/harryjo97/EHGNN) 

* **GemNet: Universal Directional Graph Neural Networks for Molecules**, *NeurIPS*, 2021. [\[pdf\]](https://arxiv.org/pdf/2106.08903.pdf) [\[code\]](https://www.daml.in.tum.de/gemnet) 

* **A 3D Generative Model for Structure-Based Drug Design**, *NeurIPS*, 2021. [\[pdf\]](https://arxiv.org/pdf/2203.10446.pdf) [\[code\]](https://github.com/luost26/3D-Generative-SBDD) 

* **E(n) Equivariant Normalizing Flows**, *NeurIPS*, 2021. [\[pdf\]](https://arxiv.org/pdf/2105.09016.pdf)  

* **Learning Space Partitions for Path Planning**, *NeurIPS*, 2021. [\[pdf\]](https://arxiv.org/pdf/2106.10544.pdf) [\[code\]](https://github.com/yangkevin2/neurips2021-lap3) 

* **Autobahn: Automorphism-based Graph Neural Nets**, *NeurIPS*, 2021. [\[pdf\]](https://arxiv.org/pdf/2103.01710.pdf) [\[code\]](https://github.com/risilab/Autobahn) 

* **Language Models Enable Zero-shot Prediction of the Effects of Mutations on Protein Function**, *NeurIPS*, 2021. [\[pdf\]](https://proceedings.neurips.cc/paper_files/paper/2021/file/f51338d736f95dd42427296047067694-Paper.pdf) [\[code\]](https://github.com/facebookresearch/esm) 

* **Do Transformers Really Perform Bad for Graph Representation?**, *NeurIPS*, 2021. [\[pdf\]](https://arxiv.org/pdf/2106.05234.pdf) [\[code\]](https://github.com/Microsoft/Graphormer) 

* **Smooth Normalizing Flows**, *NeurIPS*, 2021. [\[pdf\]](https://arxiv.org/pdf/2110.00351.pdf)  

* **Flow Network based Generative Models for Non-Iterative Diverse Candidate Generation**, *NeurIPS*, 2021. [\[pdf\]](https://arxiv.org/pdf/2106.04399.pdf) [\[code\]](https://github.com/GFNOrg/gflownet) 

* **Weisfeiler and Lehman Go Cellular: CW Networks**, *NeurIPS*, 2021. [\[pdf\]](https://arxiv.org/pdf/2106.12575.pdf)  

* **Multi-Scale Representation Learning on Proteins**, *NeurIPS*, 2021. [\[pdf\]](https://arxiv.org/pdf/2204.02337.pdf)  

* **Representation Learning on Spatial Networks**, *NeurIPS*, 2021. [\[pdf\]](https://proceedings.neurips.cc/paper_files/paper/2021/file/12e35d9186dd72fe62fd039385890b9c-Paper.pdf) [\[code\]](https://github.com/rollingstonezz/SGMP_code) 

* **Predicting Molecular Conformation via Dynamic Graph Score Matching**, *NeurIPS*, 2021. [\[pdf\]](https://proceedings.neurips.cc/paper_files/paper/2021/file/a45a1d12ee0fb7f1f872ab91da18f899-Paper.pdf)  

* **Property-Aware Relation Networks for Few-Shot Molecular Property Prediction**, *NeurIPS*, 2021. [\[pdf\]](https://arxiv.org/pdf/2107.07994.pdf)  

* **Learning Equivariant Energy Based Models with Equivariant Stein Variational Gradient Descent**, *NeurIPS*, 2021. [\[pdf\]](https://arxiv.org/pdf/2106.07832.pdf)  

* **Deep Molecular Representation Learning via Fusing Physical and Chemical Information**, *NeurIPS*, 2021. [\[pdf\]](https://arxiv.org/pdf/2112.04624.pdf)  

* **Motif-based Graph Self-Supervised Learning for Molecular Property Prediction**, *NeurIPS*, 2021. [\[pdf\]](https://arxiv.org/pdf/2110.00987.pdf) [\[code\]](https://github.com/zaixizhang/MGSSL) 

* **Rethinking Neural Operations for Diverse Tasks**, *NeurIPS*, 2021. [\[pdf\]](https://arxiv.org/pdf/2103.15798.pdf) [\[code\]](https://github.com/nick11roberts/XD) 

* **Directional Message Passing on Molecular Graphs via Synthetic Coordinates**, *NeurIPS*, 2021. [\[pdf\]](https://arxiv.org/pdf/2111.04718.pdf) [\[code\]](https://www.cs.cit.tum.de/daml/startseite/) 

* **SE(3)-Equivariant Prediction of Molecular Wavefunctions and Electronic Densities**, *NeurIPS*, 2021. [\[pdf\]](https://arxiv.org/pdf/2106.02347.pdf)  

* **Co-evolution Transformer for Protein Contact Prediction**, *NeurIPS*, 2021. [\[pdf\]](https://proceedings.neurips.cc/paper_files/paper/2021/file/770f8e448d07586afbf77bb59f698587-Paper.pdf) [\[code\]](https://github.com/microsoft/ProteinFolding/tree/main/coevolution_transformer) 

* **Amortized Tree Generation for Bottom-up Synthesis Planning and Synthesizable Molecular Design**, *ICLR*, 2022. [\[pdf\]](https://arxiv.org/pdf/2110.06389.pdf) [\[code\]](https://github.com/wenhao-gao/SynNet) 

* **Iterative Refinement Graph Neural Network for Antibody Sequence-Structure Co-design**, *ICLR*, 2022. [\[pdf\]](https://arxiv.org/pdf/2110.04624.pdf)  

* **Ab-Initio Potential Energy Surfaces by Pairing GNNs with Neural Wave Functions**, *ICLR*, 2022. [\[pdf\]](https://arxiv.org/pdf/2110.05064.pdf)  

* **Independent SE(3)-Equivariant Models for End-to-End Rigid Protein Docking**, *ICLR*, 2022. [\[pdf\]](https://arxiv.org/pdf/2111.07786.pdf) [\[code\]](https://github.com/octavian-ganea/equidock_public) 

* **Geometric and Physical Quantities Improve E(3) Equivariant Message Passing**, *ICLR*, 2022. [\[pdf\]](https://arxiv.org/pdf/2110.02905.pdf)  

* **Chemical-Reaction-Aware Molecule Representation Learning**, *ICLR*, 2022. [\[pdf\]](https://arxiv.org/pdf/2109.09888.pdf) [\[code\]](https://github.com/hwwang55/MolR) 

* **OntoProtein: Protein Pretraining With Gene Ontology Embedding**, *ICLR*, 2022. [\[pdf\]](https://arxiv.org/pdf/2201.11147.pdf) [\[code\]](https://github.com/zjunlp/OntoProtein) 

* **Simple GNN Regularisation for 3D Molecular Property Prediction & Beyond**, *ICLR*, 2022. [\[pdf\]](https://arxiv.org/pdf/2106.07971.pdf) [\[code\]](https://github.com/deepmind/deepmind-research/tree/master/ogb_lsc/pcq) 

* **Bayesian Modeling and Uncertainty Quantification for Learning to Optimize: What, Why, and How**, *ICLR*, 2022. [\[pdf\]](https://openreview.net/pdf?id=EVVadRFRgL7) [\[code\]](https://github.com/Shen-Lab/Bayesian-L2O) 

* **Spherical Message Passing for 3D Molecular Graphs**, *ICLR*, 2022. [\[pdf\]](https://openreview.net/pdf?id=givsRXsOt9r) [\[code\]](https://github.com/divelab/DIG) 

* **Crystal Diffusion Variational Autoencoder for Periodic Material Generation**, *ICLR*, 2022. [\[pdf\]](https://arxiv.org/pdf/2110.06197.pdf) [\[code\]](https://github.com/txie-93/cdvae) 

* **Spatial Graph Attention and Curiosity-driven Policy for Antiviral Drug Discovery**, *ICLR*, 2022. [\[pdf\]](https://arxiv.org/pdf/2106.02190.pdf) [\[code\]](https://github.com/yulun-rayn/DGAPN) 

* **Ancestral Protein Sequence Reconstruction using A Tree-structured Ornstein-Uhlenbeck Variational Autoencoder**, *ICLR*, 2022. [\[pdf\]](https://openreview.net/pdf?id=FZoZ7a31GCW) [\[code\]](https://github.com/LysSanzMoreta/DRAUPNIR_ASR) 

* **Differentiable Scaffolding Tree for Molecular Optimization**, *ICLR*, 2022. [\[pdf\]](https://openreview.net/pdf?id=w_drCosT76) [\[code\]](https://github.com/futianfan/DST) 

* **Pre-training Molecular Graph Representation with 3D Geometry**, *ICLR*, 2022. [\[pdf\]](https://arxiv.org/pdf/2110.07728.pdf) [\[code\]](https://github.com/chao1224/GraphMVP) 

* **An Autoregressive Flow Model for 3D Molecular Geometry Generation from Scratch**, *ICLR*, 2022. [\[pdf\]](https://openreview.net/pdf?id=C03Ajc-NS5W)  

* **Learning 3D Representations of Molecular Chirality with Invariance to Bond Rotations**, *ICLR*, 2022. [\[pdf\]](https://arxiv.org/pdf/2110.04383.pdf) [\[code\]](https://github.com/keiradams/ChIRo) 

* **Energy-Inspired Molecular Conformation Optimization**, *ICLR*, 2022. [\[pdf\]](https://openreview.net/pdf?id=7QfLW-XZTl) [\[code\]](https://github.com/guanjq/confopt_official) 

* **Graph Neural Networks with Learnable Structural and Positional Representations**, *ICLR*, 2022. [\[pdf\]](https://arxiv.org/pdf/2110.07875.pdf) [\[code\]](https://github.com/vijaydwivedi75/gnn-lspe) 

* **Learning to Extend Molecular Scaffolds with Structural Motifs**, *ICLR*, 2022. [\[pdf\]](https://arxiv.org/pdf/2103.03864.pdf) [\[code\]](https://github.com/microsoft/molecule-generation) 

* **A Benchmark for Experimental Design in Drug Discovery**, *ICLR*, 2022. [\[pdf\]](https://openreview.net/pdf?id=-w2oomO6qgc)  

* **Equivariant Graph Mechanics Networks with Constraints**, *ICLR*, 2022. [\[pdf\]](https://arxiv.org/pdf/2203.06442.pdf) [\[code\]](https://github.com/hanjq17/GMN) 

* **GeoDiff: a Geometric Diffusion Model for Molecular Conformation Generation**, *ICLR*, 2022. [\[pdf\]](https://arxiv.org/pdf/2203.02923.pdf) [\[code\]](https://github.com/MinkaiXu/GeoDiff) 

* **Top-N: Equivariant Set and Graph Generation without Exchangeability**, *ICLR*, 2022. [\[pdf\]](https://arxiv.org/pdf/2110.02096.pdf) [\[code\]](https://github.com/cvignac/Top-N) 

* **Geometric Transformers for Protein Interface Contact Prediction**, *ICLR*, 2022. [\[pdf\]](https://arxiv.org/pdf/2110.02423.pdf) [\[code\]](https://github.com/BioinfoMachineLearning/DeepInteract) 

* **Data-Efficient Graph Grammar Learning for Molecular Generation**, *ICLR*, 2022. [\[pdf\]](https://arxiv.org/pdf/2203.08031.pdf) [\[code\]](https://github.com/gmh14/data_efficient_grammar) 

* **TorchMD-NET: Equivariant Transformers for Neural Network based Molecular Potentials**, *ICLR*, 2022. [\[pdf\]](https://arxiv.org/pdf/2202.02541.pdf) [\[code\]](https://github.com/torchmd/torchmd-net) 

* **Spanning Tree-based Graph Generation for Molecules**, *ICLR*, 2022. [\[pdf\]](https://openreview.net/pdf?id=w60btE_8T2m)  

* **Learning to Predict Graphs with Fused Gromov-Wasserstein Barycenters**, *ICML*, 2022. [\[pdf\]](https://arxiv.org/pdf/2202.03813.pdf) [\[code\]](https://github.com/lmotte/graph-prediction-with-fused-gromov-wasserstein) 

* **Generative Coarse-Graining of Molecular Conformations**, *ICML*, 2022. [\[pdf\]](https://arxiv.org/pdf/2201.12176.pdf) [\[code\]](https://github.com/wwang2/CoarseGrainingVAE) 

* **Efficient Higher-order Subgraph Attribution via Message Passing**, *ICML*, 2022. [\[pdf\]](https://proceedings.mlr.press/v162/xiong22a/xiong22a.pdf) [\[code\]](https://github.com/xiong-ping/sgnn_lrp_via_mp) 

* **Unified Fourier-based Kernel and Nonlinearity Design for Equivariant Networks on Homogeneous Spaces**, *ICML*, 2022. [\[pdf\]](https://arxiv.org/pdf/2206.08362.pdf)  

* **Retroformer: Pushing the Limits of End-to-end Retrosynthesis Transformer**, *ICML*, 2022. [\[pdf\]](https://proceedings.mlr.press/v162/wan22a/wan22a.pdf) [\[code\]](https://github.com/yuewan2/Retroformer) 

* **VARSCENE: A Deep Generative Model for Realistic Scene Graph Synthesis**, *ICML*, 2022. [\[pdf\]](https://proceedings.mlr.press/v162/verma22b/verma22b.pdf) [\[code\]](https://github.com/google/sg2im) 

* **Design-Bench: Benchmarks for Data-Driven Offline Model-Based Optimization**, *ICML*, 2022. [\[pdf\]](https://arxiv.org/pdf/2202.08450.pdf)  

* **EQUIBIND: Geometric Deep Learning for Drug Binding Structure Prediction**, *ICML*, 2022. [\[pdf\]](https://arxiv.org/pdf/2202.05146.pdf) [\[code\]](https://github.com/HannesStark/EquiBind) 

* **3D Infomax improves GNNs for Molecular Property Prediction**, *ICML*, 2022. [\[pdf\]](https://arxiv.org/pdf/2110.04126.pdf) [\[code\]](https://github.com/HannesStark/3DInfomax) 

* **Accelerating Bayesian Optimization for Biological Sequence Design with Denoising Autoencoders**, *ICML*, 2022. [\[pdf\]](https://arxiv.org/pdf/2203.12742.pdf) [\[code\]](https://github.com/samuelstanton/lambo) 

* **Proximal Exploration for Model-guided Protein Sequence Design**, *ICML*, 2022. [\[pdf\]](https://proceedings.mlr.press/v162/ren22a/ren22a.pdf) [\[code\]](https://github.com/HeliXonProtein/proximal-exploration) 

* **Pocket2Mol: Efficient Molecular Sampling Based on 3D Protein Pockets**, *ICML*, 2022. [\[pdf\]](https://arxiv.org/pdf/2205.07249.pdf) [\[code\]](https://github.com/pengxingang/Pocket2Mol) 

* **Tranception: Protein Fitness Prediction with Autoregressive Transformers and Inference-time Retrieval**, *ICML*, 2022. [\[pdf\]](https://arxiv.org/pdf/2205.13760.pdf) [\[code\]](https://github.com/OATML-Markslab/Tranception) 

* **Path-Aware and Structure-Preserving Generation of Synthetically Accessible Molecules**, *ICML*, 2022. [\[pdf\]](https://proceedings.mlr.press/v162/noh22a/noh22a.pdf)  

* **SPECTRE: Spectral Conditioning Helps to Overcome the Expressivity Limits of One-shot Graph Generators**, *ICML*, 2022. [\[pdf\]](https://arxiv.org/pdf/2204.01613.pdf) [\[code\]](https://github.com/KarolisMart/SPECTRE) 

* **Generating 3D Molecules for Target Protein Binding**, *ICML*, 2022. [\[pdf\]](https://arxiv.org/pdf/2204.09410.pdf)  

* **Let Invariant Rationale Discovery Inspire Graph Contrastive Learning**, *ICML*, 2022. [\[pdf\]](https://arxiv.org/pdf/2206.07869.pdf) [\[code\]](https://github.com/lsh0520/RGCL) 

* **Query-Efficient and Scalable Black-Box Adversarial Attacks on Discrete Sequential Data via Bayesian Optimization**, *ICML*, 2022. [\[pdf\]](https://arxiv.org/pdf/2206.08575.pdf) [\[code\]](https://github.com/snu-mllab/DiscreteBlockBayesAttack) 

* **Score-based Generative Modeling of Graphs via the System of Stochastic Differential Equations**, *ICML*, 2022. [\[pdf\]](https://arxiv.org/pdf/2202.02514.pdf) [\[code\]](https://github.com/harryjo97/GDSS) 

* **Antibody-Antigen Docking and Design via Hierarchical Structure Refinement**, *ICML*, 2022. [\[pdf\]](https://proceedings.mlr.press/v162/jin22a/jin22a.pdf) [\[code\]](github.com/wengong-jin/abdockgen) 

* **Biological Sequence Design with GFlowNets**, *ICML*, 2022. [\[pdf\]](https://arxiv.org/pdf/2203.04115.pdf) [\[code\]](https://github.com/MJ10/BioSeq-GFN-AL) 

* **3DLinker: An E(3) Equivariant Variational Autoencoder for Molecular Linker Design**, *ICML*, 2022. [\[pdf\]](https://arxiv.org/pdf/2205.07309.pdf)  

* **Learning Inverse Folding from Millions of Predicted Structures**, *ICML*, 2022. [\[pdf\]](https://proceedings.mlr.press/v162/hsu22a/hsu22a.pdf)  

* **Equivariant Diffusion for Molecule Generation in 3D**, *ICML*, 2022. [\[pdf\]](https://arxiv.org/pdf/2203.17003.pdf)  

* **Deep Squared Euclidean Approximation to the Levenshtein Distance for DNA Storage**, *ICML*, 2022. [\[pdf\]](https://arxiv.org/pdf/2207.04684.pdf)  

* **How to Fill the Optimum Set? Population Gradient Descent with Harmless Diversity**, *ICML*, 2022. [\[pdf\]](https://proceedings.mlr.press/v162/gong22b/gong22b.pdf)  

* **Conformal Prediction Sets with Limited False Positives**, *ICML*, 2022. [\[pdf\]](https://proceedings.mlr.press/v162/fisch22a/fisch22a.pdf) [\[code\]](https://github.com/ajfisch/conformal-fp) 

* **LIMO: Latent Inceptionism for Targeted Molecule Generation**, *ICML*, 2022. [\[pdf\]](https://arxiv.org/pdf/2206.09010.pdf) [\[code\]](https://github.com/Rose-STL-Lab/LIMO) 

* **SE(3) Equivariant Graph Neural Networks with Complete Local Frames**, *ICML*, 2022. [\[pdf\]](https://arxiv.org/pdf/2110.14811.pdf) [\[code\]]() 

* **One-Pass Diversified Sampling with Application to Terabyte-Scale Genomic Sequence Streams**, *ICML*, 2022. [\[pdf\]](https://proceedings.mlr.press/v162/coleman22a/coleman22a.pdf)  

* **Molecular Representation Learning via Heterogeneous Motif Graph Neural Networks**, *ICML*, 2022. [\[pdf\]](https://arxiv.org/pdf/2202.00529.pdf) [\[code\]](https://github.com/ZhaoningYu1996/HM-GNN) 

* **Periodic Graph Transformers for Crystal Material Property Prediction**, *NeurIPS*, 2022. [\[pdf\]](https://arxiv.org/pdf/2209.11807.pdf) 

* **Recipe for a General, Powerful, Scalable Graph Transformer**, *NeurIPS*, 2022. [\[pdf\]](https://arxiv.org/pdf/2205.12454.pdf) [\[code\]](https://github.com/rampasek/GraphGPS) 

* **Graph Self-supervised Learning with Accurate Discrepancy Learning**, *NeurIPS*, 2022. [\[pdf\]](https://arxiv.org/pdf/2202.02989.pdf) [\[code\]](https://github.com/DongkiKim95/D-SLA) 

* **Amortized Inference for Heterogeneous Reconstruction in Cryo-EM**, *NeurIPS*, 2022. [\[pdf\]](https://arxiv.org/pdf/2210.07387.pdf)  

* **Learning Substructure Invariance for Out-of-Distribution Molecular Representations**, *NeurIPS*, 2022. [\[pdf\]](https://openreview.net/pdf?id=2nWUNTnFijm) [\[code\]](https://github.com/yangnianzu0515/MoleOOD) 

* **Modular Flows: Differential Molecular Generation**, *NeurIPS*, 2022. [\[pdf\]](https://arxiv.org/pdf/2210.06032.pdf)  

* **Sparse Fourier Backpropagation in Cryo-EM Reconstruction**, *NeurIPS*, 2022. [\[pdf\]](https://openreview.net/pdf?id=51f5sPXJD_E) [\[code\]](https://github.com/dkimanius/sbackprop) 

* **Reinforced Genetic Algorithm for Structure-based Drug Design**, *NeurIPS*, 2022. [\[pdf\]](https://arxiv.org/pdf/2211.16508.pdf) [\[code\]](https://github.com/futianfan/reinforced-genetic-algorithm) 

* **Does GNN Pretraining Help Molecular Representation?**, *NeurIPS*, 2022. [\[pdf\]](https://arxiv.org/pdf/2207.06010.pdf)  

* **Task-Agnostic Graph Explanations**, *NeurIPS*, 2022. [\[pdf\]](https://arxiv.org/pdf/2202.08335.pdf) [\[code\]](https://github.com/divelab/DIG/tree/main/dig/xgraph/TAGE/) 

* **MACE: Higher Order Equivariant Message Passing Neural Networks for Fast and Accurate Force Fields**, *NeurIPS*, 2022. [\[pdf\]](https://arxiv.org/pdf/2206.07697.pdf) [\[code\]](https://github.com/ACEsuit/mace) 

* **Gold-standard Solutions to the Schrödinger Equation using Deep Learning: How Much Physics Do We Need?**, *NeurIPS*, 2022. [\[pdf\]](https://arxiv.org/pdf/2205.09438.pdf) [\[code\]](https://github.com/mdsunivie/deeperwin) 

* **GRASP: Navigating Retrosynthetic Planning with Goal-driven Policy**, *NeurIPS*, 2022. [\[pdf\]](https://openreview.net/pdf?id=ofwkaIWFqqv)  

* **Regularized Molecular Conformation Fields**, *NeurIPS*, 2022. [\[pdf\]](https://openreview.net/pdf?id=7XCFxnG8nGS) [\[code\]](https://github.com/leowang1217/RMCF) 

* **Cross-Linked Unified Embedding for cross-modality representation learning**, *NeurIPS*, 2022. [\[pdf\]](https://openreview.net/pdf?id=Tfb73TeKnJ-) [\[code\]](https://github.com/gao-lab/GLUE) 

* **Antigen-Specific Antibody Design and Optimization with Diffusion-Based Generative Models for Protein Structures**, *NeurIPS*, 2022. [\[pdf\]](https://openreview.net/pdf?id=jSorGn2Tjg) [\[code\]](https://github.com/luost26/diffab) 

* **Equivariant Graph Hierarchy-Based Neural Networks**, *NeurIPS*, 2022. [\[pdf\]](https://arxiv.org/pdf/2202.10643.pdf) [\[code\]](https://github.com/hanjq17/EGHN) 

* **An Efficient Graph Generative Model for Navigating Ultra-large Combinatorial Synthesis Libraries**, *NeurIPS*, 2022. [\[pdf\]](https://arxiv.org/pdf/2211.04468.pdf) [\[code\]](https://github.com/AtomwiseInc/cslvae) 

* **Residual Multiplicative Filter Networks for Multiscale Reconstruction**, *NeurIPS*, 2022. [\[pdf\]](https://arxiv.org/pdf/2206.00746.pdf) [\[code\]](https://shekshaa.github.io/ResidualMFN/) 

* **TANKBind: Trigonometry-Aware Neural NetworKs for Drug-Protein Binding Structure Prediction**, *NeurIPS*, 2022. [\[pdf\]](https://openreview.net/pdf?id=MSBDFwGYwwt) [\[code\]]() 

* **ComENet: Towards Complete and Efficient Message Passing for 3D Molecular Graphs**, *NeurIPS*, 2022. [\[pdf\]](https://arxiv.org/pdf/2206.08515.pdf) 

* **Non-identifiability and the Blessings of Misspecification in Models of Molecular Fitness**, *NeurIPS*, 2022. [\[pdf\]](https://openreview.net/pdf?id=CwG-o0ind6t)  

* **On the Symmetries of the Synchronization Problem in Cryo-EM: Multi-Frequency Vector Diffusion Maps on the Projective Plane**, *NeurIPS*, 2022. [\[pdf\]](https://openreview.net/pdf?id=owDcdLGgEm)  

* **Equivariant Networks for Crystal Structures**, *NeurIPS*, 2022. [\[pdf\]](https://arxiv.org/pdf/2211.15420.pdf)  

* **Exploring Evolution-aware & -free Protein Language Models as Protein Function Predictors**, *NeurIPS*, 2022. [\[pdf\]](https://arxiv.org/pdf/2206.06583.pdf) [\[code\]](https://github.com/elttaes/Revisiting-PLMs) 

* **Diffusion-based Molecule Generation with Informative Prior Bridges**, *NeurIPS*, 2022. [\[pdf\]](https://arxiv.org/pdf/2209.00865.pdf)  

* **Local Latent Space Bayesian Optimization over Structured Inputs**, *NeurIPS*, 2022. [\[pdf\]](https://arxiv.org/pdf/2201.11872.pdf) [\[code\]](https://github.com/nataliemaus/lolbo) 

* **Unsupervised Learning of Group Invariant and Equivariant Representations**, *NeurIPS*, 2022. [\[pdf\]](https://arxiv.org/pdf/2202.07559.pdf)  

* **Symmetry-induced Disentanglement on Graphs**, *NeurIPS*, 2022. [\[pdf\]](https://openreview.net/pdf?id=4tM0P_4N8D9)  

* **So3krates: Equivariant Attention for Interactions on Arbitrary Length-scales in Molecular Systems**, *NeurIPS*, 2022. [\[pdf\]](https://arxiv.org/pdf/2205.14276.pdf) [\[code\]](https://github.com/thorben-frank/mlff) 

* **Multi-objective Deep Data Generation with Correlated Property Control**, *NeurIPS*, 2022. [\[pdf\]](https://arxiv.org/pdf/2210.01796.pdf) [\[code\]](https://github.com/shi-yu-wang/CorrVAE) 

* **ToDD: Topological Compound Fingerprinting in Computer-Aided Drug Discovery**, *NeurIPS*, 2022. [\[pdf\]](https://arxiv.org/pdf/2211.03808.pdf)  

* **Debiased, Longitudinal and Coordinated Drug Recommendation through Multi-Visit Clinic Records**, *NeurIPS*, 2022. [\[pdf\]](https://openreview.net/pdf?id=zVglD2W0EAS) [\[code\]](https://github.com/ssshddd/DrugRec) 

* **Probabilistic Transformer: Modelling Ambiguities and Distributions for RNA Folding and Molecule Design**, *NeurIPS*, 2022. [\[pdf\]](https://arxiv.org/pdf/2205.13927.pdf) [\[code\]](https://github.com/automl/ProbTransformer) 

* **Predicting Cellular Responses to Novel Drug Perturbations at a Single-Cell Resolution**, *NeurIPS*, 2022. [\[pdf\]](https://arxiv.org/pdf/2204.13545.pdf) [\[code\]](https://github.com/theislab/chemCPA) 

* **Molecule Generation by Principal Subgraph Mining and Assembling**, *NeurIPS*, 2022. [\[pdf\]](https://arxiv.org/pdf/2106.15098.pdf) [\[code\]](https://github.com/THUNLP-MT/PS-VAE) 

* **Torsional Diffusion for Molecular Conformer Generation**, *NeurIPS*, 2022. [\[pdf\]](https://arxiv.org/pdf/2206.01729.pdf) [\[code\]](https://github.com/gcorso/torsional-diffusion) 

* **Zero-Shot 3D Drug Design by Sketching and Generating**, *NeurIPS*, 2022. [\[pdf\]](https://arxiv.org/pdf/2209.13865.pdf) [\[code\]](https://github.com/longlongman/DESERT) 


